<!DOCTYPE html SYSTEM "about:legacy-compat"><html lang="en" class="win-std-desktop-val"><head>
<META http-equiv="Content-Type" content="text/html">
<title>Source View: https://www.oncotarget.com/article/2480/text/</title><meta name="robots" content="noindex,nofollow"><meta name="SortSite-IgnoreRule-Page" content="AccHtmlLinkTextReused,AccAdvancedReadingLevel,Section508-m-3,BugHtmlColgroup,BugIeCssMinMaxWidthHeight,ErrSpellingError,SeoBingTitleLength,SeoPage150k,UseGov5.1,UseGov9.2.3,UseGov10.11,UseGov11.7,UseGov14.3,UseGov17.4,UseGunningFog,W3cTitleTooLong"><link rel="icon" href="https://www-powermapper-com.azureedge.net/favicon.ico"><link rel="stylesheet" href="../Report/report-min-20200618.css"><script type="text/javascript" src="../Report/report-min-20210409.js"></script><!--[if lte IE 8]>
            <script>
                document.createElement('header');
                document.createElement('aside');
            </script>
            <![endif]--></head><body class="source"><a class="accessible" href="#content">Skip to Content</a><header><h1>
                        HTML Document
                    </h1><table><tr><th>
                        URL:
                    </th><td><a rel="nofollow" target="_top" href="https://www.oncotarget.com/article/2480/text/">https://www.oncotarget.com/article/2480/text/</a></td></tr><tr><th>
                        HTTP Status:
                    </th><td>200 OK</td></tr><tr><th>
                        MIME Type:
                    </th><td>text/html </td></tr><tr><th>
                        Last Modified:
                    </th><td>Mon, 01 Jan 1601 00:00:00 GMT</td></tr><tr><th>
                        Download Time:
                    </th><td>1 seconds
                            </td></tr><tr><th>
                        Cookies:
                    </th><td>
                                None
                            </td></tr><tr><th>
                        Size:
                    </th><td>90 KB
                        </td></tr><tr><th>
                                Depth:
                            </th><td>4 clicks from home page
                            </td></tr><tr><th>
                                Charset:
                            </th><td>utf-8</td></tr><tr><th>
                                Forms:
                            </th><td>1 forms containing 7 controls
                            </td></tr><tr><th>
                        HTTP Headers:
                    </th><td><button class="chevron toggleChevron" data-toggleid="HttpHeaders"><img width="20" height="20" alt="View Headers" id="chevHttpHeaders" src="../Report/chevron-down.svg"></button>&nbsp;<span>11&nbsp;headers</span><div class="links" id="expand-HttpHeaders">CF-Cache-Status: DYNAMIC<br>CF-RAY: 80d7000baaf22b9b-LAX<br>Cache-Control: no-store<br>Connection: keep-alive<br>Date: Wed, 27 Sep 2023 21:57:04 GMT<br>NEL: {"success_fraction":0,"report_to":"cf-nel","max_age":604800}<br>Report-To: {"endpoints":[{"url":"https:\/\/a.nel.cloudflare.com\/report\/v3?s=491ffpV2s9%2Bw3qjSp%2BwZrNn%2FBqMMRTVKjePvcNnu%2FAiJe44jyREMF9gq%2BNe4q4bvuWa5mLUKxZ5rsAIMVPUHHW2g3G9gLIbqp09E%2BbsHQ3CzyzQ5Ftqx7ejq2qb3cSfH5HS5fug%3D"}],"group":"cf-nel","max_age":604800}<br>Server: cloudflare<br>Transfer-Encoding: chunked<br>X-Powered-By: PHP/5.3.29<br>alt-svc: h3=":443"; ma=86400<br></div></td></tr><tr><th>
                        Links In:
                    </th><td><button class="chevron toggleChevron" data-toggleid="LinksIn"><img width="20" height="20" alt="View Links" id="chevLinksIn" src="../Report/chevron-down.svg"></button>&nbsp;<span>1&nbsp;pages</span><div class="links" id="expand-LinksIn"><a class="viewsource" rel="nofollow" target="_top" href="https://content.montefioreeinstein.org/cancer/research/cdtmi/cdtmi-research" data-pageid="6651" data-line="0" data-col="0">https://content.montefioreeinstein.org/cancer/research/cdtmi/cdtmi-research</a><br></div></td></tr><tr><td>&nbsp;</td><td>&nbsp;</td></tr><tr class="middle"><th><img alt="OK" width="20" height="20" src="../Report/bench-better.svg">
                                Issues:
                            </th><td>
                                No issues found
                            </td></tr></table></header><main id="content" class="pre"><p><code>1</code><code>          </code></p><p><code>2</code><code>          </code></p><p><code>3</code><code>&lt;?xml version="1.0" encoding="UTF-8"?&gt;</code></p><p><code>4</code><code>&lt;!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"&gt;</code></p><p><code>5</code><code>&lt;html xmlns="http://www.w3.org/1999/xhtml" lang="en"&gt;</code></p><p><code>6</code><code>&lt;head&gt;</code></p><p><code>7</code><code>	</code></p><p><code>8</code><code>          </code></p><p><code>9</code><code>    </code></p><p><code>10</code><code>    &lt;script&gt;</code></p><p><code>11</code><code>     // &lt;!-- Google Tag Manager --&gt;</code></p><p><code>12</code><code>    (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':</code></p><p><code>13</code><code>    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],</code></p><p><code>14</code><code>    j=d.createElement(s),dl=l!='dataLayer'?'&amp;l='+l:'';j.async=true;j.src=</code></p><p><code>15</code><code>    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);</code></p><p><code>16</code><code>    })(window,document,'script','dataLayer','GTM-KXPN39');</code></p><p><code>17</code><code>    //&lt;!-- End Google Tag Manager --&gt;</code></p><p><code>18</code><code>    &lt;/script&gt;</code></p><p><code>19</code><code>    </code></p><p><code>20</code><code>          </code></p><p><code>21</code><code>    &lt;title&gt;Characterization of single disseminated prostate cancer cells reveals tumor cell heterogeneity and identifies dormancy associated pathways | Oncotarget&lt;/title&gt;</code></p><p><code>22</code><code>          </code></p><p><code>23</code><code>    </code></p><p><code>24</code><code>          </code></p><p><code>25</code><code>	&lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;</code></p><p><code>26</code><code>          </code></p><p><code>27</code><code>    &lt;meta name="description" content="https://doi.org/10.18632/oncotarget.2480 Lisly Chéry, Hung-Ming Lam, Ilsa Coleman, Bryce Lakely, Roger Coleman, Sandy Larson, Julio A. Aguirre-Ghiso, Jing Xia, Roman Gulati, Peter S. Nelson, Bruce..." /&gt;</code></p><p><code>28</code><code>          </code></p><p><code>29</code><code>    </code></p><p><code>30</code><code>	</code></p><p><code>31</code><code>&lt;script type="application/ld+json"&gt;{"@context":"https:\/\/schema.org","@graph":{"@type":"Article","license":"https:\/\/creativecommons.org\/licenses\/by\/3.0\/","publisher":{"@type":"Organization","name":"Oncotarget","logo":{"@type":"ImageObject","url":"https:\/\/www.oncotarget.com\/images\/oncotarget-logo-square.png","width":1200,"height":1200}},"dateModified":"2014-11-05","mainEntityOfPage":{"@type":"WebPage","@id":"https:\/\/www.oncotarget.com\/article\/2480\/text\/"},"headline":"Characterization of single disseminated prostate cancer cells reveals tumor cell heterogeneity and identifi","articleSection":"Research Papers","sameAs":"https:\/\/doi.org\/10.18632\/oncotarget.2480","image":"https:\/\/www.oncotarget.com\/index.php?journal=oncotarget&amp;page=article&amp;op=viewFile&amp;path%5B%5D=2480&amp;path%5B%5D=4418&amp;path%5B%5D=29158","datePublished":"2014-09-16","author":[{"@type":"Person","name":"Ch\u00e9ry, Lisly"},{"@type":"Person","name":"Lam, Hung-Ming"},{"@type":"Person","name":"Coleman, Ilsa"},{"@type":"Person","name":"Lakely, Bryce"},{"@type":"Person","name":"Coleman, Roger"},{"@type":"Person","name":"Larson, Sandy"},{"@type":"Person","name":"Aguirre-Ghiso, Julio A."},{"@type":"Person","name":"Xia, Jing"},{"@type":"Person","name":"Gulati, Roman"},{"@type":"Person","name":"Nelson, Peter S."},{"@type":"Person","name":"Montgomery, Bruce"},{"@type":"Person","name":"Lange, Paul"},{"@type":"Person","name":"Snyder, Linda A."},{"@type":"Person","name":"Vessella, Robert L."},{"@type":"Person","name":"Morrissey, Colm"}],"description":"Cancer dormancy refers to the prolonged clinical disease-free time between removal of the primary tumor and recurrence, which is common in prostate cancer (PCa), breast cancer, esophageal cancer, and other cancers. PCa disseminated tumor cells (DTC) are detected in both patients with no evidence of disease (NED) and advanced disease (ADV). However, the molecular and cellular nature of DTC is unknown. We performed a first-in-field study of single DTC transcriptomic analyses in cancer patients to identify a molecular signature associated with cancer dormancy. We profiled eighty-five individual EpCAM+\/CD45- cells from the bone marrow of PCa patients with NED or ADV. We analyzed 44 DTC with high prostate-epithelial signatures, and eliminated 41 cells with high erythroid signatures and low prostate epithelial signatures. DTC were clustered into 3 groups: NED, ADV_1, and ADV_2, in which the ADV_1 group presented a distinct gene expression pattern associated with the p38 stress activated kinase pathway. Additionally, DTC from the NED group were enriched for a tumor dormancy signature associated with head and neck squamous carcinoma and breast cancer. This study provides the first clinical evidence of the p38 pathway as a potential biomarker for early recurrence and an attractive target for therapeutic intervention."}}&lt;/script&gt;</code></p><p><code>32</code><code>          </code></p><p><code>33</code><code>          </code></p><p><code>34</code><code>	&lt;link rel="icon" href="https://www.oncotarget.com/favicon.ico" /&gt;</code></p><p><code>35</code><code>          </code></p><p><code>36</code><code>	</code></p><p><code>37</code><code>	&lt;meta name="dc.creator" content="Lisly Chéry"/&gt;</code></p><p><code>38</code><code>	&lt;meta name="dc.creator" content="Hung-Ming Lam"/&gt;</code></p><p><code>39</code><code>	&lt;meta name="dc.creator" content="Ilsa Coleman"/&gt;</code></p><p><code>40</code><code>	&lt;meta name="dc.creator" content="Bryce Lakely"/&gt;</code></p><p><code>41</code><code>	&lt;meta name="dc.creator" content="Roger Coleman"/&gt;</code></p><p><code>42</code><code>	&lt;meta name="dc.creator" content="Sandy Larson"/&gt;</code></p><p><code>43</code><code>	&lt;meta name="dc.creator" content="Julio A. Aguirre-Ghiso"/&gt;</code></p><p><code>44</code><code>	&lt;meta name="dc.creator" content="Jing Xia"/&gt;</code></p><p><code>45</code><code>	&lt;meta name="dc.creator" content="Roman Gulati"/&gt;</code></p><p><code>46</code><code>	&lt;meta name="dc.creator" content="Peter S. Nelson"/&gt;</code></p><p><code>47</code><code>	&lt;meta name="dc.creator" content="Bruce Montgomery"/&gt;</code></p><p><code>48</code><code>	&lt;meta name="dc.creator" content="Paul Lange"/&gt;</code></p><p><code>49</code><code>	&lt;meta name="dc.creator" content="Linda A. Snyder"/&gt;</code></p><p><code>50</code><code>	&lt;meta name="dc.creator" content="Robert L. Vessella"/&gt;</code></p><p><code>51</code><code>	&lt;meta name="dc.creator" content="Colm Morrissey"/&gt;</code></p><p><code>52</code><code>	&lt;meta name="dc.date" scheme="ISO8601" content="2014-09-16"/&gt;</code></p><p><code>53</code><code>	&lt;meta name="dc.date.created" scheme="ISO8601" content="2014-09-16"/&gt;</code></p><p><code>54</code><code>    &lt;meta name="dc.date.datesubmitted" scheme="ISO8601" content="2014-09-16"/&gt;</code></p><p><code>55</code><code>		&lt;meta name="dc.date.issued" scheme="ISO8601" content="2014-10-30"/&gt;</code></p><p><code>56</code><code>	&lt;meta name="dc.date.modified" scheme="ISO8601" content="2014-11-05"/&gt;</code></p><p><code>57</code><code>	&lt;meta name="dc.description" xml:lang="en" content="  //     Lisly Ch&amp;amp;eacute;ry 1,* , Hung-Ming Lam 1,* , Ilsa Coleman 2 , Bryce Lakely 1 , Roger Coleman 2 , Sandy Larson 1 , Julio A. Aguirre-Ghiso 3 , Jing Xia 4 , Roman Gulati 4 , Peter S. Nelson 2,5 , Bruce Montgomery 5 , Paul Lange 6,1 , Linda A. Snyder 7 , Robert L. Vessella 6,1  and Colm Morrissey 1     1  Department of Urology, University of Washington, Seattle, WA    2  Divison of Human Biology, Fred Hutchinson Cancer Research Center, Seattle, WA    3  Division of Hematology and Oncology, Department of Medicine and Department of Otolaryngology, Tisch Cancer Institute, Black Family Stem Cell Institute, Ichan School of Medicine at Mount Sinai, New York, NY    4  Divison of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA    5  Department of Medicine, University of Washington, Seattle, WA    6  Department of Veterans Affairs Medical Center, Seattle, WA    7  Janssen Research and Development, LLC, Spring House, PA    *  These authors contributed equally to this work   Correspondence:   Colm Morrissey , email:  //       Keywords : Prostate cancer, dormancy, metastasis, p38, gene expression    Received : July 03, 2014  Accepted : September 15, 2014  Published : September 16, 2014    Abstract   Cancer dormancy refers to the prolonged clinical disease-free time between removal of the primary tumor and recurrence, which is common in prostate cancer (PCa), breast cancer, esophageal cancer, and other cancers. PCa disseminated tumor cells (DTC) are detected in both patients with no evidence of disease (NED) and advanced disease (ADV). However, the molecular and cellular nature of DTC is unknown. We performed a first-in-field study of single DTC transcriptomic analyses in cancer patients to identify a molecular signature associated with cancer dormancy. We profiled eighty-five individual EpCAM + /CD45 -  cells from the bone marrow of PCa patients with NED or ADV. We analyzed 44 DTC with high prostate-epithelial signatures, and eliminated 41 cells with high erythroid signatures and low prostate epithelial signatures. DTC were clustered into 3 groups: NED, ADV_1, and ADV_2, in which the ADV_1 group presented a distinct gene expression pattern associated with the p38 stress activated kinase pathway. Additionally, DTC from the NED group were enriched for a tumor dormancy signature associated with head and neck squamous carcinoma and breast cancer. This study provides the first clinical evidence of the p38 pathway as a potential biomarker for early recurrence and an attractive target for therapeutic intervention. "/&gt;</code></p><p><code>58</code><code>	&lt;meta name="dc.format" scheme="IMT" content="application/pdf"/&gt;</code></p><p><code>59</code><code>	&lt;meta name="dc.format" scheme="IMT" content="text/html"/&gt;</code></p><p><code>60</code><code>	&lt;meta name="dc.identifier" content="https://doi.org/10.18632/oncotarget.2480"/&gt;</code></p><p><code>61</code><code>	&lt;meta name="dc.identifier.pagenumber" content="9939-9951"/&gt;</code></p><p><code>62</code><code>	&lt;meta name="citation_doi" content="10.18632/oncotarget.2480"/&gt;</code></p><p><code>63</code><code>	&lt;meta name="dc.identifier.uri" content="https://www.oncotarget.com/article/2480/"/&gt;</code></p><p><code>64</code><code>	&lt;meta name="dc.language" scheme="ISO639-1" content=""/&gt;</code></p><p><code>65</code><code>	&lt;meta name="dc.source" content="Oncotarget"/&gt;</code></p><p><code>66</code><code>	&lt;meta name="dc.source.issn" content="1949-2553"/&gt;</code></p><p><code>67</code><code>&lt;meta name="dc.source.uri" content="https://www.oncotarget.com/"/&gt;</code></p><p><code>68</code><code>	&lt;meta name="dc.source.issue" content="20"/&gt;</code></p><p><code>69</code><code>&lt;meta name="dc.source.volume" content="5"/&gt;</code></p><p><code>70</code><code>					&lt;meta name="dc.title" content="Characterization of single disseminated prostate cancer cells reveals tumor cell heterogeneity and identifies dormancy associated pathways"/&gt;</code></p><p><code>71</code><code>	</code></p><p><code>72</code><code>&lt;meta name="dc.type" content="Text.Serial.Journal"/&gt;</code></p><p><code>73</code><code>&lt;meta name="dc.type.articletype" content="Research Papers"/&gt;</code></p><p><code>74</code><code>          </code></p><p><code>75</code><code>          </code></p><p><code>76</code><code>&lt;!-- OGP --&gt;</code></p><p><code>77</code><code>	&lt;meta property="og:type"  content="article" /&gt;</code></p><p><code>78</code><code>	&lt;meta property="og:title" content="Characterization of single disseminated prostate cancer cells reveals tumor cell heterogeneity and identifies dormancy associated pathways | Oncotarget" /&gt;</code></p><p><code>79</code><code>				&lt;meta property="og:description" content="https://doi.org/10.18632/oncotarget.2480 Lisly Chéry, Hung-Ming Lam, Ilsa Coleman, Bryce Lakely, Roger Coleman, Sandy Larson, Julio A. Aguirre-Ghiso, Jing Xia, Roman Gulati, Peter S. Nelson, Bruce..."/&gt;</code></p><p><code>80</code><code>		&lt;meta property="og:url"   content="https://www.oncotarget.com/article/2480/text/" /&gt;</code></p><p><code>81</code><code>	&lt;meta property="og:image" content="https://www.oncotarget.com/index.php?journal=oncotarget&amp;page=article&amp;op=viewFile&amp;path%5B%5D=2480&amp;path%5B%5D=4418&amp;path%5B%5D=29158"  /&gt;</code></p><p><code>82</code><code>&lt;!-- /OGP --&gt;&lt;!-- Twitter Card --&gt;</code></p><p><code>83</code><code>&lt;meta name="twitter:card" content="summary" /&gt;</code></p><p><code>84</code><code>&lt;meta name="twitter:site" content="@Oncotarget" /&gt;</code></p><p><code>85</code><code>&lt;meta name="twitter:title" content="Characterization of single disseminated prostate cancer cells reveals tumor cell heterogeneity and identifies dormancy associated pathways | Oncotarget" /&gt;</code></p><p><code>86</code><code>  &lt;meta name="twitter:description" content="https://doi.org/10.18632/oncotarget.2480 Lisly Chéry, Hung-Ming Lam, Ilsa Coleman, Bryce Lakely, Roger Coleman, Sandy Larson, Julio A. Aguirre-Ghiso, Jing Xia, Roman Gulati, Peter S. Nelson, Bruce..."/&gt;</code></p><p><code>87</code><code>&lt;meta name="twitter:image" content="https://www.oncotarget.com/index.php?journal=oncotarget&amp;page=article&amp;op=viewFile&amp;path%5B%5D=2480&amp;path%5B%5D=4418&amp;path%5B%5D=29158"  /&gt;</code></p><p><code>88</code><code>&lt;!-- /Twitter Card --&gt;		&lt;meta name="gs_meta_revision" content="1.1" /&gt;</code></p><p><code>89</code><code>	&lt;meta name="citation_journal_title" content="Oncotarget"/&gt;</code></p><p><code>90</code><code>	&lt;meta name="citation_issn" content="1949-2553"/&gt;</code></p><p><code>91</code><code>	&lt;meta name="citation_author" content="Lisly Chéry"/&gt;</code></p><p><code>92</code><code>		</code></p><p><code>93</code><code>	&lt;meta name="citation_author" content="Hung-Ming Lam"/&gt;</code></p><p><code>94</code><code>		</code></p><p><code>95</code><code>	&lt;meta name="citation_author" content="Ilsa Coleman"/&gt;</code></p><p><code>96</code><code>		</code></p><p><code>97</code><code>	&lt;meta name="citation_author" content="Bryce Lakely"/&gt;</code></p><p><code>98</code><code>		</code></p><p><code>99</code><code>	&lt;meta name="citation_author" content="Roger Coleman"/&gt;</code></p><p><code>100</code><code>		</code></p><p><code>101</code><code>	&lt;meta name="citation_author" content="Sandy Larson"/&gt;</code></p><p><code>102</code><code>		</code></p><p><code>103</code><code>	&lt;meta name="citation_author" content="Julio A. Aguirre-Ghiso"/&gt;</code></p><p><code>104</code><code>		</code></p><p><code>105</code><code>	&lt;meta name="citation_author" content="Jing Xia"/&gt;</code></p><p><code>106</code><code>		</code></p><p><code>107</code><code>	&lt;meta name="citation_author" content="Roman Gulati"/&gt;</code></p><p><code>108</code><code>		</code></p><p><code>109</code><code>	&lt;meta name="citation_author" content="Peter S. Nelson"/&gt;</code></p><p><code>110</code><code>		</code></p><p><code>111</code><code>	&lt;meta name="citation_author" content="Bruce Montgomery"/&gt;</code></p><p><code>112</code><code>		</code></p><p><code>113</code><code>	&lt;meta name="citation_author" content="Paul Lange"/&gt;</code></p><p><code>114</code><code>		</code></p><p><code>115</code><code>	&lt;meta name="citation_author" content="Linda A. Snyder"/&gt;</code></p><p><code>116</code><code>		</code></p><p><code>117</code><code>	&lt;meta name="citation_author" content="Robert L. Vessella"/&gt;</code></p><p><code>118</code><code>		</code></p><p><code>119</code><code>	&lt;meta name="citation_author" content="Colm Morrissey"/&gt;</code></p><p><code>120</code><code>		</code></p><p><code>121</code><code>	&lt;meta name="citation_authors" content="Chéry, Lisly; Lam, Hung-Ming; Coleman, Ilsa; Lakely, Bryce; Coleman, Roger; Larson, Sandy; Aguirre-Ghiso, Julio A.; Xia, Jing; Gulati, Roman; Nelson, Peter S.; Montgomery, Bruce; Lange, Paul; Snyder, Linda A.; Vessella, Robert L.; Morrissey, Colm"/&gt;</code></p><p><code>122</code><code>	&lt;meta name="citation_title" content="Characterization of single disseminated prostate cancer cells reveals tumor cell heterogeneity and identifies dormancy associated pathways"/&gt;</code></p><p><code>123</code><code>	&lt;meta name="citation_publication_date" content="2014-09-16"/&gt;</code></p><p><code>124</code><code>	&lt;meta name="citation_volume" content="5"/&gt;</code></p><p><code>125</code><code>			&lt;meta name="citation_issue" content="20"/&gt;</code></p><p><code>126</code><code>		&lt;meta name="citation_firstpage" content="9939"/&gt;</code></p><p><code>127</code><code>	&lt;meta name="citation_lastpage" content="9951"/&gt;</code></p><p><code>128</code><code>	&lt;meta name="citation_abstract_html_url" content="https://www.oncotarget.com/article/2480/"/&gt;</code></p><p><code>129</code><code>				</code></p><p><code>130</code><code>        	&lt;meta name="citation_pdf_url" content="https://www.oncotarget.com/article/2480/pdf/" /&gt;</code></p><p><code>131</code><code>                    	&lt;meta name="citation_fulltext_html_url" content="https://www.oncotarget.com/article/2480/text/"/&gt;</code></p><p><code>132</code><code>    	&lt;meta name="citation_publisher" content="Impact Journals"/&gt;</code></p><p><code>133</code><code>	</code></p><p><code>134</code><code>          </code></p><p><code>135</code><code>	&lt;link rel="stylesheet" href="https://www.oncotarget.com/styles/paper.css" type="text/css" /&gt;</code></p><p><code>136</code><code>	&lt;link rel="stylesheet" href="https://www.oncotarget.com/jquery.bubblepopup.v2.3.1.css"type="text/css" /&gt;</code></p><p><code>137</code><code>          </code></p><p><code>138</code><code>	&lt;link rel="stylesheet" href="https://www.oncotarget.com/lib/pkp/styles/pkp.css" type="text/css" /&gt;</code></p><p><code>139</code><code>	&lt;link rel="stylesheet" href="https://www.oncotarget.com/lib/pkp/styles/common.css" type="text/css" /&gt;</code></p><p><code>140</code><code>	&lt;link rel="stylesheet" href="https://www.oncotarget.com/styles/common.css" type="text/css" /&gt;</code></p><p><code>141</code><code>	&lt;link rel="stylesheet" href="https://www.oncotarget.com/styles/articleView.css" type="text/css" /&gt;</code></p><p><code>142</code><code>	&lt;link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/foundicons/3.0.0/foundation-icons.min.css" type="text/css"/&gt;</code></p><p><code>143</code><code>	&lt;link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.0.6/css/all.css"&gt;</code></p><p><code>144</code><code>	&lt;link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css"&gt; </code></p><p><code>145</code><code>          </code></p><p><code>146</code><code>	&lt;link rel="stylesheet" href="https://www.oncotarget.com/style/agingTemp.min.css" type="text/css" /&gt;</code></p><p><code>147</code><code>          </code></p><p><code>148</code><code>	</code></p><p><code>149</code><code>	</code></p><p><code>150</code><code>	</code></p><p><code>151</code><code>          </code></p><p><code>152</code><code>	&lt;link rel="stylesheet" href="https://www.oncotarget.com/styles/sidebar.css" type="text/css" /&gt;	&lt;link rel="stylesheet" href="https://www.oncotarget.com/styles/leftSidebar.css" type="text/css" /&gt;	&lt;link rel="stylesheet" href="https://www.oncotarget.com/styles/rightSidebar.css?v=1.0.1" type="text/css" /&gt;	&lt;link rel="stylesheet" href="https://www.oncotarget.com/styles/bothSidebars.css" type="text/css" /&gt;</code></p><p><code>153</code><code>			&lt;link rel="stylesheet" href="https://www.oncotarget.com/public/site/sitestyle.css" type="text/css" /&gt;</code></p><p><code>154</code><code>	</code></p><p><code>155</code><code>	</code></p><p><code>156</code><code>	&lt;script&gt;</code></p><p><code>157</code><code>	  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){</code></p><p><code>158</code><code>	  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),</code></p><p><code>159</code><code>	  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)</code></p><p><code>160</code><code>	  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');</code></p><p><code>161</code><code>	  ga('create', 'UA-74807910-1', 'auto');</code></p><p><code>162</code><code>	  ga('send', 'pageview');</code></p><p><code>163</code><code>	&lt;/script&gt;</code></p><p><code>164</code><code>	</code></p><p><code>165</code><code>          </code></p><p><code>166</code><code>    &lt;!-- altmetric embed --&gt;</code></p><p><code>167</code><code>    &lt;script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'&gt;&lt;/script&gt;</code></p><p><code>168</code><code>	&lt;!-- Base Jquery --&gt;</code></p><p><code>169</code><code>		&lt;script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"&gt;&lt;/script&gt;</code></p><p><code>170</code><code>	&lt;link rel="stylesheet" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/smoothness/jquery-ui.css"&gt;</code></p><p><code>171</code><code>	&lt;script src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"&gt;&lt;/script&gt;</code></p><p><code>172</code><code>	</code></p><p><code>173</code><code>	&lt;script type="text/javascript" src="https://www.oncotarget.com/lib/pkp/js/jquery.cookie.js"&gt;&lt;/script&gt;</code></p><p><code>174</code><code>	&lt;script type="text/javascript" src="https://www.oncotarget.com/lib/pkp/js/fontController.min.js" &gt;&lt;/script&gt;</code></p><p><code>175</code><code>	&lt;script type="text/javascript" src="https://www.oncotarget.com/lib/pkp/js/general.min.js"&gt;&lt;/script&gt;</code></p><p><code>176</code><code>          </code></p><p><code>177</code><code>	</code></p><p><code>178</code><code>	&lt;script language="javascript" type="text/javascript" src="https://www.oncotarget.com/js/articleView.min.js"&gt;&lt;/script&gt;</code></p><p><code>179</code><code>	&lt;script language="javascript" type="text/javascript" src="https://www.oncotarget.com/js/pdfobject.js"&gt;&lt;/script&gt;</code></p><p><code>180</code><code>          </code></p><p><code>181</code><code>          </code></p><p><code>182</code><code>	</code></p><p><code>183</code><code>	&lt;script&gt;</code></p><p><code>184</code><code>	function SelectText(element) {</code></p><p><code>185</code><code>		var text = document.getElementById(element);</code></p><p><code>186</code><code>		if ($.browser.msie) {</code></p><p><code>187</code><code>			var range = document.body.createTextRange();</code></p><p><code>188</code><code>			range.moveToElementText(text);</code></p><p><code>189</code><code>			range.select();</code></p><p><code>190</code><code>		} else if ($.browser.mozilla || $.browser.opera) {</code></p><p><code>191</code><code>			var selection = window.getSelection();</code></p><p><code>192</code><code>			var range = document.createRange();</code></p><p><code>193</code><code>			range.selectNodeContents(text);</code></p><p><code>194</code><code>			selection.removeAllRanges();</code></p><p><code>195</code><code>			selection.addRange(range);</code></p><p><code>196</code><code>		} else if ($.browser.safari) {</code></p><p><code>197</code><code>			var selection = window.getSelection();</code></p><p><code>198</code><code>			selection.setBaseAndExtent(text, 0, text, 1);</code></p><p><code>199</code><code>		}</code></p><p><code>200</code><code>		document.getElementById(element).scrollIntoView();</code></p><p><code>201</code><code>	}</code></p><p><code>202</code><code>	&lt;/script&gt;</code></p><p><code>203</code><code>	&lt;script type="text/javascript" src="/js/email.min.js"&gt;&lt;/script&gt;</code></p><p><code>204</code><code>	&lt;script type="text/javascript" src="/js/pops.min.js"&gt;&lt;/script&gt;</code></p><p><code>205</code><code>	&lt;script type="text/javascript" src="/jquery.bubblepopup.v2.3.1.min.js"&gt;&lt;/script&gt;</code></p><p><code>206</code><code>	&lt;script type="text/javascript" src="/js/menu.min.js"&gt;&lt;/script&gt;</code></p><p><code>207</code><code>	</code></p><p><code>208</code><code>	</code></p><p><code>209</code><code>&lt;/head&gt;</code></p><p><code>210</code><code>&lt;body&gt;</code></p><p><code>211</code><code>&lt;!-- Google Tag Manager (noscript) --&gt;</code></p><p><code>212</code><code>&lt;noscript&gt;&lt;iframe src="https://www.googletagmanager.com/ns.html?id=GTM-KXPN39"</code></p><p><code>213</code><code>height="0" width="0" style="display:none;visibility:hidden"&gt;&lt;/iframe&gt;&lt;/noscript&gt;</code></p><p><code>214</code><code>&lt;!-- End Google Tag Manager (noscript) --&gt;</code></p><p><code>215</code><code>&lt;div id="container"&gt;</code></p><p><code>216</code><code>&lt;div id="fade" class="black_overlay"&gt;&lt;/div&gt;</code></p><p><code>217</code><code>&lt;div id="header"&gt;</code></p><p><code>218</code><code>	&lt;a href="/"&gt;&lt;div id="home-clickable-area"&gt;&lt;/div&gt;&lt;/a&gt;</code></p><p><code>219</code><code>	&lt;div class="nav-content"&gt;</code></p><p><code>220</code><code>		&lt;ul class="socialList segregateSubmit"&gt;</code></p><p><code>221</code><code>			&lt;li class="social-links"&gt;&lt;a href="https://www.facebook.com/Oncotarget/" target="_blank"&gt;&lt;i class="fa-brands fa-facebook-square"&gt;&lt;/i&gt;&lt;/a&gt;&lt;/li&gt;</code></p><p><code>222</code><code>			&lt;li class="social-links"&gt;&lt;a href="https://twitter.com/Oncotarget" target="_blank"&gt;&lt;i class="fa-brands fa-x-twitter"&gt;&lt;/i&gt;&lt;/a&gt;&lt;/li&gt;</code></p><p><code>223</code><code>			&lt;li class="social-links"&gt;&lt;a href="https://www.instagram.com/oncotargetjrnl/" target="_blank"&gt;&lt;i class="fa-brands fa-instagram"&gt;&lt;/i&gt;&lt;/a&gt;&lt;/li&gt;</code></p><p><code>224</code><code>			&lt;li class="social-links"&gt;&lt;a href="https://www.linkedin.com/company/oncotarget" target="_blank"&gt;&lt;i class="fa-brands fa-linkedin"&gt;&lt;/i&gt;&lt;/a&gt;&lt;/li&gt;</code></p><p><code>225</code><code>			&lt;li class="social-links"&gt;&lt;a href="https://www.youtube.com/@OncotargetJournal" target="_blank"&gt;&lt;i class="fa-brands fa-youtube-square"&gt;&lt;/i&gt;&lt;/a&gt;&lt;/li&gt;</code></p><p><code>226</code><code>			&lt;li class="social-links"&gt;&lt;a href="https://www.reddit.com/user/Oncotarget" target="_blank"&gt;&lt;i class="fa-brands fa-reddit-square"&gt;&lt;/i&gt;&lt;/a&gt;&lt;/li&gt;</code></p><p><code>227</code><code>		&lt;/ul&gt;</code></p><p><code>228</code><code>	</code></p><p><code>229</code><code>		&lt;span class="sidebar-toggle-mobile icon icon-menu"&gt;&lt;/span&gt;</code></p><p><code>230</code><code>		&lt;span id="mobile-search" class="icon icon-search hide-on-large-only"&gt;&lt;/span&gt;</code></p><p><code>231</code><code>	&lt;/div&gt;</code></p><p><code>232</code><code>	</code></p><p><code>233</code><code>	&lt;small class="issnLabel"&gt;Online ISSN: 1949-2553&lt;/small&gt;</code></p><p><code>234</code><code>	&lt;div class="nav-search"&gt;</code></p><p><code>235</code><code>		&lt;form method="get" action="/index.php"&gt;</code></p><p><code>236</code><code>			&lt;input type="hidden" name="journal" value="oncotarget" /&gt;</code></p><p><code>237</code><code>			&lt;input type="hidden" name="page" value="pages" /&gt;</code></p><p><code>238</code><code>			&lt;input type="hidden" name="op" value="view" /&gt;</code></p><p><code>239</code><code>			&lt;input type="hidden" name="path[]" value="elasticsearch" /&gt;</code></p><p><code>240</code><code>			&lt;input type="hidden" name="searchPage" value="1" /&gt;</code></p><p><code>241</code><code>	</code></p><p><code>242</code><code>			&lt;label&gt;Search:&lt;/label&gt;&lt;br/&gt;</code></p><p><code>243</code><code>			&lt;input type="text" name="general"&gt;</code></p><p><code>244</code><code>			&lt;input type="submit" style="display:none"&gt;</code></p><p><code>245</code><code>		&lt;/form&gt;</code></p><p><code>246</code><code>	&lt;/div&gt;</code></p><p><code>247</code><code>&lt;/div&gt;</code></p><p><code>248</code><code>          </code></p><p><code>249</code><code>          </code></p><p><code>250</code><code>&lt;div id="clear"&gt;</code></p><p><code>251</code><code>&lt;p&gt;Oncotarget&lt;/p&gt;</code></p><p><code>252</code><code>&lt;/div&gt;</code></p><p><code>253</code><code>          </code></p><p><code>254</code><code>          </code></p><p><code>255</code><code>&lt;div id="leftSidebar"&gt;</code></p><p><code>256</code><code>	</code></p><p><code>257</code><code> </code></p><p><code>258</code><code> &lt;style&gt;</code></p><p><code>259</code><code> .anchor-no-underline,</code></p><p><code>260</code><code> .anchor-no-underline:link,</code></p><p><code>261</code><code> .anchor-no-underline:visited,</code></p><p><code>262</code><code> .anchor-no-underline:hover,</code></p><p><code>263</code><code> .anchor-no-underline:active {</code></p><p><code>264</code><code>     text-decoration: none;</code></p><p><code>265</code><code> }</code></p><p><code>266</code><code> </code></p><p><code>267</code><code>  a.subscribe-button {</code></p><p><code>268</code><code>    -webkit-appearance: button;</code></p><p><code>269</code><code>    -moz-appearance: button;</code></p><p><code>270</code><code>    appearance: button;</code></p><p><code>271</code><code>    text-decoration: none;</code></p><p><code>272</code><code>    border: 0 none;</code></p><p><code>273</code><code>    border-radius: 4px;</code></p><p><code>274</code><code>    color: #FFFFFF;</code></p><p><code>275</code><code>    cursor: pointer;</code></p><p><code>276</code><code>    display: inline-block;</code></p><p><code>277</code><code>    font-weight: bold;</code></p><p><code>278</code><code>    font-size:1em;</code></p><p><code>279</code><code>    height: 32px;</code></p><p><code>280</code><code>    line-height: 32px;</code></p><p><code>281</code><code>    padding: 0 22px;</code></p><p><code>282</code><code>    margin-top:5px;</code></p><p><code>283</code><code>    margin-bottom:20px;</code></p><p><code>284</code><code>    text-align: center;</code></p><p><code>285</code><code>    text-decoration: none;</code></p><p><code>286</code><code>    vertical-align: top;</code></p><p><code>287</code><code>    white-space: nowrap;</code></p><p><code>288</code><code>    width: auto;</code></p><p><code>289</code><code>    background-color:#761118;</code></p><p><code>290</code><code>  }</code></p><p><code>291</code><code> &lt;/style&gt;</code></p><p><code>292</code><code> </code></p><p><code>293</code><code>          </code></p><p><code>294</code><code>	&lt;div class="block" id="sidebarNavigation"&gt;</code></p><p><code>295</code><code>	&lt;span class="blockTitle" style="display:none;"&gt;Journal Content&lt;/span&gt;</code></p><p><code>296</code><code>	&lt;!--Custom --&gt;</code></p><p><code>297</code><code>	&lt;ul class="menu" id="navmenu"&gt;</code></p><p><code>298</code><code>		&lt;li id="home"&gt;&lt;a href="/"&gt;Home&lt;/a&gt;&lt;/li&gt;</code></p><p><code>299</code><code>		&lt;li id="board"&gt;&lt;a href="/editorial-board/"&gt;Editorial Board&lt;/a&gt;&lt;/li&gt;</code></p><p><code>300</code><code>		&lt;li&gt;&lt;a href="http://oncotarget.msubmit.net"&gt;Submission&lt;/a&gt;&lt;/li&gt;</code></p><p><code>301</code><code>					&lt;li id="currentiss"&gt;&lt;a href="/archive/v14/"&gt;Current Volume&lt;/a&gt;&lt;/li&gt;</code></p><p><code>302</code><code>						&lt;li id="archives"&gt;&lt;a href="/archive/"&gt;Archive&lt;/a&gt;&lt;/li&gt;</code></p><p><code>303</code><code>        &lt;li id="scientific_integrity"&gt;&lt;a style="color: #CC0000;" href="/scientific_integrity/"&gt;Scientific Integrity&lt;/a&gt;&lt;/li&gt;</code></p><p><code>304</code><code>		&lt;li id="about"&gt;&lt;a href="/editorial-policies/"&gt;Editorial Policies&lt;/a&gt;&lt;/li&gt;</code></p><p><code>305</code><code>		&lt;li id="ethics"&gt;&lt;a href="/ethics-statement/"&gt;Publication Ethics Statements&lt;/a&gt;&lt;/li&gt;</code></p><p><code>306</code><code>		&lt;li id="ot_special_collections"&gt;&lt;a href="/collections/"&gt;Special Collections&lt;/a&gt;&lt;/li&gt;</code></p><p><code>307</code><code>		&lt;li id="ot_videos"&gt;&lt;a href="/videos/interview/"&gt;Interviews with Outstanding Authors&lt;/a&gt;&lt;/li&gt;</code></p><p><code>308</code><code>		&lt;li id="ot_news"&gt;&lt;a href="/news/pr/"&gt;Oncotarget In The News&lt;/a&gt;&lt;/li&gt;</code></p><p><code>309</code><code>		&lt;li id="ot_conferences"&gt;&lt;a href="/news/conferences/"&gt;Sponsored Conferences&lt;/a&gt;&lt;/li&gt;</code></p><p><code>310</code><code>		&lt;li id="search"&gt;&lt;a href="/search/"&gt;Search&lt;/a&gt;&lt;/li&gt;</code></p><p><code>311</code><code>		&lt;li id="contact"&gt;&lt;a href="/contact/"&gt;Contact Information&lt;/a&gt;</code></p><p><code>312</code><code>		</code></p><p><code>313</code><code>								&lt;/ul&gt;</code></p><p><code>314</code><code>	&lt;!-- end custom--&gt;</code></p><p><code>315</code><code>	&lt;/div&gt;</code></p><p><code>316</code><code>          </code></p><p><code>317</code><code>	&lt;div class="block secblock topredblock"&gt;</code></p><p><code>318</code><code>		&lt;div class="a_new_domain_link"&gt;</code></p><p><code>319</code><code>			&lt;p&gt;</code></p><p><code>320</code><code>				Oncotarget (a primarily oncology-focused, peer-reviewed, open access journal) aims to maximize research impact through insightful peer-review; eliminate borders between specialties by linking different fields of oncology, cancer research and biomedical sciences; and foster application of basic and clinical science.</code></p><p><code>321</code><code>			&lt;/p&gt;</code></p><p><code>322</code><code>			&lt;p&gt;</code></p><p><code>323</code><code>				Its scope is unique. The term "oncotarget" encompasses all molecules, pathways, cellular functions, cell types, and even tissues that can be viewed as targets relevant to cancer as well as other diseases. &lt;b&gt;The term was introduced in the inaugural Editorial&lt;/b&gt;, &lt;a class="anchor-no-underline" href="https://www.oncotarget.com/article/101/"&gt;Introducing Oncotarget&lt;/a&gt;.</code></p><p><code>324</code><code>			&lt;/p&gt;</code></p><p><code>325</code><code>			&lt;p&gt;</code></p><p><code>326</code><code>				As of January 1, 2022, &lt;em&gt;Oncotarget&lt;/em&gt; has shifted to a continuous publishing model. Papers will now be published continuously within yearly volumes in their final and complete form and then quickly released to Pubmed.</code></p><p><code>327</code><code>			&lt;/p&gt;</code></p><p><code>328</code><code>		&lt;/div&gt;</code></p><p><code>329</code><code>		&lt;div class="a_new_domain_link"&gt;&lt;/div&gt;</code></p><p><code>330</code><code>	&lt;/div&gt;</code></p><p><code>331</code><code>          </code></p><p><code>332</code><code>          </code></p><p><code>333</code><code>	</code></p><p><code>334</code><code>	&lt;div class="block secblock topredblock"&gt;</code></p><p><code>335</code><code>		&lt;p class="navbar-section"&gt;&lt;span style="font-size:13px;font-weight:700;margin-left:0.3em;"&gt;&lt;/span&gt;&lt;/p&gt;</code></p><p><code>336</code><code>		&lt;h4 class="navbar-section"&gt;Publication Alerts&lt;/h4&gt;</code></p><p><code>337</code><code>          </code></p><p><code>338</code><code>		</code></p><p><code>339</code><code>		&lt;div style="padding: 10px;"&gt;</code></p><p><code>340</code><code>			Subscribe to receive alerts once a paper has been published by Oncotarget.</code></p><p><code>341</code><code>		&lt;/div&gt;</code></p><p><code>342</code><code>		&lt;div style="text-align:center;"&gt;</code></p><p><code>343</code><code>			&lt;a href="/subscribe/" class="subscribe-button"&gt;Subscribe&lt;/a&gt;</code></p><p><code>344</code><code>		&lt;/div&gt;</code></p><p><code>345</code><code>          </code></p><p><code>346</code><code>		</code></p><p><code>347</code><code>			&lt;/div&gt;</code></p><p><code>348</code><code>          </code></p><p><code>349</code><code>  </code></p><p><code>350</code><code>  </code></p><p><code>351</code><code>          </code></p><p><code>352</code><code>  </code></p><p><code>353</code><code>          </code></p><p><code>354</code><code>    &lt;div style="padding:10px 4px 10px 10px;display:block;" class="block secblock topredblock"&gt;</code></p><p><code>355</code><code>        &lt;p style="font-size: 1.1em;"&gt;Impact Journals, LLC is the publisher of Oncotarget: &lt;a class="anchor-no-underline" target="_blank" href="https://www.impactjournals.com/"&gt;www.impactjournals.com&lt;/a&gt;.&lt;/p&gt;</code></p><p><code>356</code><code>        &lt;p style="font-size: 1.1em;"&gt;Impact Journals is a member of the &lt;a class="anchor-no-underline" target="_blank" href="https://docs.google.com/document/d/1iKsQN2I4S45SdWox4wuPPW8sQpS9B7h1TK1ThJUOGYo/edit"&gt;Wellcome Trust List of Compliant Publishers&lt;/a&gt;.&lt;/p&gt;</code></p><p><code>357</code><code>        &lt;p style="font-size: 1.1em;"&gt;Impact Journals is a member of the &lt;a class="anchor-no-underline" target="_blank" href="https://www.sspnet.org/"&gt;Society for Scholarly Publishing&lt;/a&gt;.&lt;/p&gt;</code></p><p><code>358</code><code>    &lt;/div&gt;</code></p><p><code>359</code><code>          </code></p><p><code>360</code><code>	</code></p><p><code>361</code><code>		&lt;div class="block secblock topredblock" style="padding:10px 4px 10px 10px;display:block;"&gt;</code></p><p><code>362</code><code>		&lt;p&gt;On December 23, 2022, Oncotarget server experienced a DDoS attack. As a result, Oncotarget site was inaccessible for a few hours. Oncotarget team swiftly dealt with the situation and took it under control.</code></p><p><code>363</code><code>		This malicious action will be reported to the FBI.&lt;/p&gt;</code></p><p><code>364</code><code>	&lt;/div&gt;</code></p><p><code>365</code><code>          </code></p><p><code>366</code><code>			</code></p><p><code>367</code><code>          </code></p><p><code>368</code><code>          </code></p><p><code>369</code><code>          </code></p><p><code>370</code><code>          </code></p><p><code>371</code><code>          </code></p><p><code>372</code><code>&lt;/div&gt;</code></p><p><code>373</code><code>          </code></p><p><code>374</code><code>          </code></p><p><code>375</code><code>&lt;div id="main" style="width:75%; clear:none; float:right;"&gt;</code></p><p><code>376</code><code>          </code></p><p><code>377</code><code>          </code></p><p><code>378</code><code>&lt;div id="content"&gt;</code></p><p><code>379</code><code>&lt;style scoped&gt;</code></p><p><code>380</code><code>          </code></p><p><code>381</code><code>	.rhs_banners {</code></p><p><code>382</code><code>		display: table;</code></p><p><code>383</code><code>		float: right;</code></p><p><code>384</code><code>		margin-right: 10px;</code></p><p><code>385</code><code>	}</code></p><p><code>386</code><code>	.addthis_container-row {</code></p><p><code>387</code><code>		display: table-row;</code></p><p><code>388</code><code>	}</code></p><p><code>389</code><code>	</code></p><p><code>390</code><code>	.addthis_container {</code></p><p><code>391</code><code>		display: table-cell;</code></p><p><code>392</code><code>		padding:8px;</code></p><p><code>393</code><code>	}</code></p><p><code>394</code><code>	.article-notice .banner-main {</code></p><p><code>395</code><code>		font-size: 16px;</code></p><p><code>396</code><code>	}</code></p><p><code>397</code><code>	.article-notice-investigation .banner-main strong {</code></p><p><code>398</code><code>		font-size: 16px;</code></p><p><code>399</code><code>	}</code></p><p><code>400</code><code>          </code></p><p><code>401</code><code>	.article-notice-investigation .banner-main {</code></p><p><code>402</code><code>		height: 45px;</code></p><p><code>403</code><code>	}</code></p><p><code>404</code><code>          </code></p><p><code>405</code><code>	span.investigation-warning {</code></p><p><code>406</code><code>		/*margin-left: 0 !important;*/</code></p><p><code>407</code><code>	}</code></p><p><code>408</code><code>          </code></p><p><code>409</code><code>					</code></p><p><code>410</code><code>			div.OncoFigure {</code></p><p><code>411</code><code>				overflow:scroll;</code></p><p><code>412</code><code>			}</code></p><p><code>413</code><code>			div.OncoFigure p.TableHeading{</code></p><p><code>414</code><code>				overflow:scroll;</code></p><p><code>415</code><code>			}</code></p><p><code>416</code><code>		</code></p><p><code>417</code><code>	&lt;/style&gt;</code></p><p><code>418</code><code>          </code></p><p><code>419</code><code>          </code></p><p><code>420</code><code>          </code></p><p><code>421</code><code>	&lt;style scoped&gt;</code></p><p><code>422</code><code>	</code></p><p><code>423</code><code>		#articleTitle {</code></p><p><code>424</code><code>			left: 0 !important;</code></p><p><code>425</code><code>		}</code></p><p><code>426</code><code>		.__dimensions_badge_embed__ {</code></p><p><code>427</code><code>			margin-bottom: 5px;</code></p><p><code>428</code><code>		}</code></p><p><code>429</code><code>		.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle .__dimensions_Badge_Image {</code></p><p><code>430</code><code>			width: 50px;</code></p><p><code>431</code><code>			height: 50px;</code></p><p><code>432</code><code>		}</code></p><p><code>433</code><code>		.__dimensions_badge_embed__ &gt; .__dimensions_Link, .__dimensions_badge_embed__ &gt; .__dimensions_Link:hover {</code></p><p><code>434</code><code>			margin-right: 0.75em;</code></p><p><code>435</code><code>		}</code></p><p><code>436</code><code>		.__dimensions_badge_embed__ .__dimensions_Badge_style_small_circle .__db_score {</code></p><p><code>437</code><code>			font-size: 28px;</code></p><p><code>438</code><code>		}</code></p><p><code>439</code><code>		.altmetric-embed &gt; a {</code></p><p><code>440</code><code>			padding-right: 7px;</code></p><p><code>441</code><code>		}</code></p><p><code>442</code><code>		.special_collections_banner-row {</code></p><p><code>443</code><code>			display: table-row;</code></p><p><code>444</code><code>		}</code></p><p><code>445</code><code>          </code></p><p><code>446</code><code>		.special_collections_banner {</code></p><p><code>447</code><code>			display: table-cell;</code></p><p><code>448</code><code>			width: 165px;</code></p><p><code>449</code><code>			padding: 1px;</code></p><p><code>450</code><code>			border: 1px solid #e8e8e8;</code></p><p><code>451</code><code>		}</code></p><p><code>452</code><code>		.special_collections_banner img {</code></p><p><code>453</code><code>			width: 165px;</code></p><p><code>454</code><code>			max-height: 100px;</code></p><p><code>455</code><code>		}</code></p><p><code>456</code><code>	</code></p><p><code>457</code><code>				</code></p><p><code>458</code><code>			.metrics-extendable {</code></p><p><code>459</code><code>				min-height: 15px;</code></p><p><code>460</code><code>			}</code></p><p><code>461</code><code>		</code></p><p><code>462</code><code>			&lt;/style&gt;</code></p><p><code>463</code><code>          </code></p><p><code>464</code><code>	&lt;script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"&gt;&lt;/script&gt;</code></p><p><code>465</code><code>          </code></p><p><code>466</code><code>          </code></p><p><code>467</code><code>	&lt;div class="issue-info" style="display: none"&gt;</code></p><p><code>468</code><code>		Published in &lt;a href="/archive/v5/i20/"&gt;Oncotarget V5I20&lt;/a&gt;, &lt;span class="issue-date"&gt;Oct 30, 2014&lt;/span&gt;</code></p><p><code>469</code><code>	&lt;/div&gt;</code></p><p><code>470</code><code>          </code></p><p><code>471</code><code>          </code></p><p><code>472</code><code>          </code></p><p><code>473</code><code>          </code></p><p><code>474</code><code>          </code></p><p><code>475</code><code>			&lt;h4 class="tocSectionTitle"&gt;Research Papers:&lt;/h4&gt;</code></p><p><code>476</code><code>          </code></p><p><code>477</code><code>					</code></p><p><code>478</code><code>          </code></p><p><code>479</code><code>		&lt;div class = "titleAndBadge"&gt;</code></p><p><code>480</code><code>						</code></p><p><code>481</code><code>				&lt;div class="__dimensions_badge_embed__" data-doi="10.18632/oncotarget.2480" data-legend="hover-left" data-style="small_circle" data-hide-zero-citations="true"&gt;&lt;/div&gt;</code></p><p><code>482</code><code>										&lt;div data-badge-type="donut" data-doi="10.18632/oncotarget.2480" data-template="oncotarget"</code></p><p><code>483</code><code>					data-hide-no-mentions="true" data-hide-less-than="12" data-condensed='true' data-link-target="_blank" data-badge-popover="left" class="altmetric-embed"&gt;</code></p><p><code>484</code><code>				&lt;/div&gt;</code></p><p><code>485</code><code>						&lt;h1 id="articleTitle" class="tocArticleTitle"&gt;Characterization of single disseminated prostate cancer cells reveals tumor cell heterogeneity and identifies dormancy associated pathways&lt;/h1&gt;</code></p><p><code>486</code><code>		&lt;/div&gt;</code></p><p><code>487</code><code>          </code></p><p><code>488</code><code>					&lt;div class="rhs_banners"&gt;</code></p><p><code>489</code><code>			&lt;!-- start AddThis --&gt;</code></p><p><code>490</code><code>							&lt;div class="addthis_container-row"&gt;</code></p><p><code>491</code><code>															</code></p><p><code>492</code><code>							&lt;script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML'&gt;&lt;/script&gt;</code></p><p><code>493</code><code>						</code></p><p><code>494</code><code>						</code></p><p><code>495</code><code>							&lt;script type="text/javascript"&gt;</code></p><p><code>496</code><code>								var addthis_pub = '{$sharingUserName}';</code></p><p><code>497</code><code>							&lt;/script&gt;</code></p><p><code>498</code><code>						</code></p><p><code>499</code><code>										&lt;div class="addthis_container"&gt;</code></p><p><code>500</code><code>						&lt;a href="http://www.addthis.com/bookmark.php"</code></p><p><code>501</code><code>							onmouseover="return addthis_open(this, '', '', 'Characterization of single disseminated prostate cancer cells reveals tumor cell heterogeneity and identifies dormancy associated pathways')"</code></p><p><code>502</code><code>							onmouseout="addthis_close()" onclick="return addthis_sendto()"&gt;</code></p><p><code>503</code><code>							&lt;img src="https://s7.addthis.com/static/btn/sm-share-en.gif" width="83" height="16" border="0" alt="Bookmark and Share" style="border:0;padding:0" /&gt;</code></p><p><code>504</code><code>						&lt;/a&gt;</code></p><p><code>505</code><code>						&lt;script type="text/javascript" src="https://s7.addthis.com/js/200/addthis_widget.js"&gt;&lt;/script&gt;</code></p><p><code>506</code><code>					&lt;/div&gt;</code></p><p><code>507</code><code>								&lt;/div&gt;</code></p><p><code>508</code><code>						&lt;!-- end AddThis --&gt;</code></p><p><code>509</code><code>          </code></p><p><code>510</code><code>							&lt;div class="special_collections_banner-row"&gt;</code></p><p><code>511</code><code>					&lt;div class="special_collections_banner"&gt;</code></p><p><code>512</code><code>						&lt;a href="/collections/prostate-cancer/"&gt;&lt;img src="https://www.oncotarget.com/images/covers/special_collection/thumbnails/oncotarget-prostate-cancer-collection-cover.png" alt="Special Collection ToC" /&gt;&lt;/a&gt;</code></p><p><code>513</code><code>					&lt;/div&gt;</code></p><p><code>514</code><code>				&lt;/div&gt;</code></p><p><code>515</code><code>						&lt;/div&gt;</code></p><p><code>516</code><code>          </code></p><p><code>517</code><code>			&lt;div class="galleria"&gt;</code></p><p><code>518</code><code>				&lt;p&gt;</code></p><p><code>519</code><code>																		&lt;a href="https://www.oncotarget.com/article/2480/pdf/" class="file" target="_parent"&gt;PDF&lt;/a&gt;&amp;nbsp;|&amp;nbsp;																																&lt;a href="https://www.oncotarget.com/article/2480/text/" class="file" target="_parent"&gt;HTML&lt;/a&gt;																		</code></p><p><code>520</code><code>											&lt;span&gt;&amp;nbsp;|&amp;nbsp;&amp;nbsp;&lt;a href="javascript:openRTWindow('https://www.oncotarget.com/index.php?journal=oncotarget&amp;amp;page=rt&amp;amp;op=suppFiles&amp;amp;path%5B%5D=2480&amp;amp;path%5B%5D=4418');" class="file"&gt;Supplementary Files&lt;/a&gt;&lt;/span&gt;</code></p><p><code>521</code><code>					</code></p><p><code>522</code><code>											&lt;span&gt;&amp;nbsp;|&amp;nbsp;&amp;nbsp;&lt;a href="javascript:openRTWindow('https://www.oncotarget.com/index.php?journal=oncotarget&amp;amp;page=rt&amp;amp;op=captureCite&amp;amp;path%5B%5D=2480&amp;amp;path%5B%5D=4418');" class="file"&gt;How to cite&lt;/a&gt;&lt;/span&gt;</code></p><p><code>523</code><code>					</code></p><p><code>524</code><code>					</code></p><p><code>525</code><code>					</code></p><p><code>526</code><code>					</code></p><p><code>527</code><code>					</code></p><p><code>528</code><code>									&lt;/p&gt;</code></p><p><code>529</code><code>          </code></p><p><code>530</code><code>				&lt;p style="margin-top:6px;"&gt;</code></p><p><code>531</code><code>					&lt;div class="article-issue-info"&gt;</code></p><p><code>532</code><code>																			Oncotarget. 2014; 5:9939-9951. &lt;a href="https://doi.org/10.18632/oncotarget.2480"&gt;https://doi.org/10.18632/oncotarget.2480&lt;/a&gt;</code></p><p><code>533</code><code>											&lt;/div&gt;</code></p><p><code>534</code><code>				&lt;/p&gt;</code></p><p><code>535</code><code>			&lt;/div&gt;</code></p><p><code>536</code><code>          </code></p><p><code>537</code><code>							&lt;div class="metrics metrics-extendable"&gt;</code></p><p><code>538</code><code>					&lt;p class="metrics-p"&gt;</code></p><p><code>539</code><code>						&lt;span class="metrics-bold" style="float:left;margin-right: 5px;"&gt;Metrics: &lt;/span&gt;</code></p><p><code>540</code><code>						&lt;span style="float:left;"&gt;</code></p><p><code>541</code><code>																																		&lt;span class="metrics-bold"&gt;PDF&lt;/span&gt; 3662 views										&lt;span class="metrics-bold metrics-divider"&gt;&amp;nbsp;|&amp;nbsp;&amp;nbsp;&lt;/span&gt;</code></p><p><code>542</code><code>																																																																														&lt;span class="metrics-bold"&gt;HTML&lt;/span&gt; 4284 views											&lt;span class="metrics-bold metrics-divider"&gt;&amp;nbsp;|&amp;nbsp;&amp;nbsp;&lt;/span&gt;</code></p><p><code>543</code><code>																																								&lt;/span&gt;</code></p><p><code>544</code><code>						&lt;span class="metrics-explained"&gt;?&lt;/span&gt;</code></p><p><code>545</code><code>						&lt;span&gt;&amp;nbsp;&lt;/span&gt;</code></p><p><code>546</code><code>					&lt;/p&gt;</code></p><p><code>547</code><code>				&lt;/div&gt;</code></p><p><code>548</code><code>					</code></p><p><code>549</code><code>		&lt;div id="primitiveHeader"&gt;</code></p><p><code>550</code><code>			&lt;p class="author" id="titleAuthors"  align="justify" &gt;</code></p><p><code>551</code><code>				&lt;span id="p17256" class="p17256 author special"&gt;Lisly Chéry&lt;/span&gt;&lt;span id="p17257" class="p17257 author special"&gt;, Hung-Ming Lam&lt;/span&gt;&lt;span id="p17258" class="p17258 author special"&gt;, Ilsa Coleman&lt;/span&gt;&lt;span id="p17259" class="p17259 author special"&gt;, Bryce Lakely&lt;/span&gt;&lt;span id="p17260" class="p17260 author special"&gt;, Roger Coleman&lt;/span&gt;&lt;span id="p17261" class="p17261 author special"&gt;, Sandy Larson&lt;/span&gt;&lt;span id="p17262" class="p17262 author special"&gt;, Julio A. Aguirre-Ghiso&lt;/span&gt;&lt;span id="p17263" class="p17263 author special"&gt;, Jing Xia&lt;/span&gt;&lt;span id="p17264" class="p17264 author special"&gt;, Roman Gulati&lt;/span&gt;&lt;span id="p17265" class="p17265 author special"&gt;, Peter S. Nelson&lt;/span&gt;&lt;span id="p17266" class="p17266 author special"&gt;, Bruce Montgomery&lt;/span&gt;&lt;span id="p17267" class="p17267 author special"&gt;, Paul Lange&lt;/span&gt;&lt;span id="p17268" class="p17268 author special"&gt;, Linda A. Snyder&lt;/span&gt;&lt;span id="p17269" class="p17269 author special"&gt;, Robert L. Vessella&lt;/span&gt;&lt;span id="p17270" class="p17270 author special"&gt; and&amp;nbsp;Colm Morrissey&amp;nbsp;&lt;a href="&amp;#109;&amp;#97;&amp;#105;&amp;#108;&amp;#116;&amp;#111;&amp;#58;%63%6d%6f%72%72%69%73%73@%75.%77%61%73%68%69%6e%67%74%6f%6e.%65%64%75" class="miniemail"&gt;&amp;#x5f;&lt;/a&gt;&lt;/span&gt;			 &lt;/p&gt;</code></p><p><code>552</code><code>		&lt;/div&gt;</code></p><p><code>553</code><code>          </code></p><p><code>554</code><code>					&lt;div id="articleAbstract"&gt;</code></p><p><code>555</code><code>				 				&lt;h2 class="body-heading" id="abstractHeader"&gt;Abstract&lt;/h2&gt;</code></p><p><code>556</code><code>				&lt;div class="body-bodytext"&gt;</code></p><p><code>557</code><code>					&lt;div id="extendedInfo"&gt;&lt;script type="text/javascript"&gt;// &lt;![CDATA[</code></p><p><code>558</code><code>$('.header-date').hide();$('#titleAuthors').hide();$('#abstractHeader').hide();</code></p><p><code>559</code><code>// ]]&gt;&lt;/script&gt; &lt;p class="Authors"&gt;Lisly Ch&amp;eacute;ry&lt;span class="AuthorSuper"&gt;1,*&lt;/span&gt;, Hung-Ming Lam&lt;span class="AuthorSuper"&gt;1,*&lt;/span&gt;, Ilsa Coleman&lt;span class="AuthorSuper"&gt;2&lt;/span&gt;, Bryce Lakely&lt;span class="AuthorSuper"&gt;1&lt;/span&gt;, Roger Coleman&lt;span class="AuthorSuper"&gt;2&lt;/span&gt;, Sandy Larson&lt;span class="AuthorSuper"&gt;1&lt;/span&gt;, Julio A. Aguirre-Ghiso&lt;span class="AuthorSuper"&gt;3&lt;/span&gt;, Jing Xia&lt;span class="AuthorSuper"&gt;4&lt;/span&gt;, Roman Gulati&lt;span class="AuthorSuper"&gt;4&lt;/span&gt;, Peter S. Nelson&lt;span class="AuthorSuper"&gt;2,5&lt;/span&gt;, Bruce Montgomery&lt;span class="AuthorSuper"&gt;5&lt;/span&gt;, Paul Lange&lt;span class="AuthorSuper"&gt;6,1&lt;/span&gt;, Linda A. Snyder&lt;span class="AuthorSuper"&gt;7&lt;/span&gt;, Robert L. Vessella&lt;span class="AuthorSuper"&gt;6,1&lt;/span&gt; and Colm Morrissey&lt;span class="AuthorSuper"&gt;1&lt;/span&gt;&lt;/p&gt; &lt;p class="org"&gt;&lt;span class="AuthorNumber"&gt;1 &lt;/span&gt;Department of Urology, University of Washington, Seattle, WA&lt;/p&gt; &lt;p class="org"&gt;&lt;span class="AuthorNumber"&gt;2 &lt;/span&gt;Divison of Human Biology, Fred Hutchinson Cancer Research Center, Seattle, WA&lt;/p&gt; &lt;p class="org"&gt;&lt;span class="AuthorNumber"&gt;3 &lt;/span&gt;Division of Hematology and Oncology, Department of Medicine and Department of Otolaryngology, Tisch Cancer Institute, Black Family Stem Cell Institute, Ichan School of Medicine at Mount Sinai, New York, NY&lt;/p&gt; &lt;p class="org"&gt;&lt;span class="AuthorNumber"&gt;4 &lt;/span&gt;Divison of Public Health Sciences, Fred Hutchinson Cancer Research Center, Seattle, WA&lt;/p&gt; &lt;p class="org"&gt;&lt;span class="AuthorNumber"&gt;5 &lt;/span&gt;Department of Medicine, University of Washington, Seattle, WA&lt;/p&gt; &lt;p class="org"&gt;&lt;span class="AuthorNumber"&gt;6 &lt;/span&gt;Department of Veterans Affairs Medical Center, Seattle, WA&lt;/p&gt; &lt;p class="org"&gt;&lt;span class="AuthorNumber"&gt;7 &lt;/span&gt;Janssen Research and Development, LLC, Spring House, PA&lt;/p&gt; &lt;p class="orglast"&gt;&lt;span class="AuthorNumber"&gt;* &lt;/span&gt;These authors contributed equally to this work&lt;/p&gt; &lt;p class="pub-info"&gt;Correspondence:&lt;/p&gt; &lt;p id="vcard"&gt;Colm Morrissey , email: &lt;script type="text/javascript"&gt;// &lt;![CDATA[</code></p><p><code>560</code><code>mail("cmorriss","u.washington",14,"")</code></p><p><code>561</code><code>// ]]&gt;&lt;/script&gt;&lt;/p&gt; &lt;p class="Keywords"&gt;&lt;span class="CorespondanceBold"&gt;Keywords&lt;/span&gt;: Prostate cancer, dormancy, metastasis, p38, gene expression&lt;/p&gt; &lt;p class="Date"&gt;&lt;span class="CorespondanceBold"&gt;Received&lt;/span&gt;: July 03, 2014 &lt;span class="CorespondanceBold"&gt;Accepted&lt;/span&gt;: September 15, 2014 &lt;span class="CorespondanceBold"&gt;Published&lt;/span&gt;: September 16, 2014&lt;/p&gt;&lt;/div&gt; &lt;h2 id="abstractHeaderNew" class="body-heading"&gt;Abstract&lt;/h2&gt; &lt;p class="BodyText"&gt;Cancer dormancy refers to the prolonged clinical disease-free time between removal of the primary tumor and recurrence, which is common in prostate cancer (PCa), breast cancer, esophageal cancer, and other cancers. PCa disseminated tumor cells (DTC) are detected in both patients with no evidence of disease (NED) and advanced disease (ADV). However, the molecular and cellular nature of DTC is unknown. We performed a first-in-field study of single DTC transcriptomic analyses in cancer patients to identify a molecular signature associated with cancer dormancy. We profiled eighty-five individual EpCAM&lt;span class="BodySuper"&gt;+&lt;/span&gt;/CD45&lt;span class="BodySuper"&gt;-&lt;/span&gt; cells from the bone marrow of PCa patients with NED or ADV. We analyzed 44 DTC with high prostate-epithelial signatures, and eliminated 41 cells with high erythroid signatures and low prostate epithelial signatures. DTC were clustered into 3 groups: NED, ADV_1, and ADV_2, in which the ADV_1 group presented a distinct gene expression pattern associated with the p38 stress activated kinase pathway. Additionally, DTC from the NED group were enriched for a tumor dormancy signature associated with head and neck squamous carcinoma and breast cancer. This study provides the first clinical evidence of the p38 pathway as a potential biomarker for early recurrence and an attractive target for therapeutic intervention.&lt;/p&gt;</code></p><p><code>562</code><code>				&lt;/div&gt;</code></p><p><code>563</code><code>			&lt;/div&gt;</code></p><p><code>564</code><code>			&lt;hr/&gt;</code></p><p><code>565</code><code>		</code></p><p><code>566</code><code>		&lt;!-- begin &lt;html&gt; --&gt;</code></p><p><code>567</code><code>     &lt;div id="x001065R1.html" xml:lang="en-US"&gt;      &lt;div class="removeMe"&gt;      &lt;h2 class="Heading"&gt;Introduction&lt;/h2&gt;       &lt;p class="BodyText"&gt;An estimated 29,480 men will die from PCa in the United States in 2014 [&lt;a href='#bib1' onclick="SelectText('bib1');return true;"&gt;1&lt;/a&gt;]. The majority of patients who present with clinically localized prostate cancer (PCa) proceed to primary treatment with curative intent [&lt;a href='#bib2' onclick="SelectText('bib2');return true;"&gt;2&lt;/a&gt;]. Patients diagnosed with organ-confined disease have decreased recurrence and increased overall survival when compared to patients with cancer that has metastasized [&lt;a href='#bib3' onclick="SelectText('bib3');return true;"&gt;3&lt;/a&gt;, &lt;a href='#bib4' onclick="SelectText('bib4');return true;"&gt;4&lt;/a&gt;]. Disseminated disease has a poor prognosis as no curative treatments exist. &lt;/p&gt;       &lt;p class="BodyText"&gt;Not all cells that escape from the primary tumor form clinical metastases. Primary tumor cells that enter the vasculature are called circulating tumor cells (CTC). The majority of CTC are unable to establish in peripheral tissues and will die in the vasculature [&lt;a href='#bib5' onclick="SelectText('bib5');return true;"&gt;5&lt;/a&gt;-&lt;a href='#bib8' onclick="SelectText('bib8');return true;"&gt;8&lt;/a&gt;]. When CTC are able to exit the vasculature and establish residency in distal tissues such as the lung, liver, and bone marrow (BM), they are termed disseminated tumor cells (DTC). Several solid tumors, including PCa, have been postulated to have DTC that can remain in a state of dormancy for an extended period of time [&lt;a href='#bib9' onclick="SelectText('bib9');return true;"&gt;9&lt;/a&gt;-&lt;a href='#bib11' onclick="SelectText('bib11');return true;"&gt;11&lt;/a&gt;]. In PCa, these dormant DTC can become active years after treatment of the primary tumor as exhibited by late PSA recurrences [&lt;a href='#bib12' onclick="SelectText('bib12');return true;"&gt;12&lt;/a&gt;].&lt;/p&gt;       &lt;p class="BodyText"&gt;Dormant DTC pose a difficult clinical problem. First, they are difficult to detect and they may be present in patients that are asymptomatic or in those with active disease. Second, in both patient groups, dormant DTC are likely resistant to most treatments, which target cells undergoing replication and/or have identifiable and targetable signaling pathways. Third, dormant DTC do not express any known biomarker that is detectable via non-invasive or minimally invasive techniques such as urine collection or peripheral blood draw. Fourth, there are not any FDA approved tests that can readily identify and isolate dormant DTC. &lt;/p&gt;       &lt;p class="BodyText"&gt;The characterization of dormant DTC and identification of the pathways involved in tumor cell dormancy are needed to determine which patients are likely to have dormant DTC that have the potential to eventually progress to clinical metastases, and to identify potential targets for therapeutic intervention. In this study, our objectives were to identify DTC in patients with PCa, characterize DTC at the single-cell level, analyze the heterogeneity of DTC within and among patients, and identify markers of tumor cell dormancy.&lt;/p&gt;       &lt;h2 class="Heading"&gt;Results&lt;/h2&gt;       &lt;h3 class="SubHeading"&gt;Sample analysis&lt;/h3&gt;       &lt;p class="BodyText"&gt;BM was drawn from five patients diagnosed with PCa and no evidence of disease following primary treatment (NED) and six patients with metastatic disease or biochemical recurrence following primary treatment (ADV). The characteristics of the patients in each group are displayed in Supplemental Table 1. One hundred and five individual EpCAM&lt;span class="BodySuper"&gt;+&lt;/span&gt;/CD45&lt;span class="BodySuper"&gt;-&lt;/span&gt; cells were isolated from the BM aspirates of the 11 PCa patients. Briefly, BM aspirates were placed over Ficoll-Isopaque, the mononuclear cell layer containing DTC underwent immunomagnetic negative selection to eliminate leukocytes, megakaryocytes, and platelets and then positive selection for epithelial cells. The enriched population was subjected to immunostaining and EpCAM&lt;span class="BodySuper"&gt;+&lt;/span&gt;/CD45&lt;span class="BodySuper"&gt;-&lt;/span&gt; cells were isolated using a micromanipulator. Quality control for the genetic material isolated and amplified from isolated single cell DTC was performed as previously detailed [&lt;a href='#bib13' onclick="SelectText('bib13');return true;"&gt;13&lt;/a&gt;]. Twenty cells were removed due to more than 25 percent of probes having missing values or to poor microarray hybridization signals, ending up with 85 cells for gene expression analyses [Supplemental Figure 1&lt;/a&gt;]. Thirty-nine cells were analyzed from NED patients and 46 cells were analyzed from ADV patients. Each cell that was isolated was given an EpCAM intensity [Supplemental Figure 2&lt;/a&gt;].&lt;/p&gt;      &lt;div class="OncoFigure"&gt;        &lt;div class="image"&gt;&lt;img src="https://www.oncotarget.com/index.php?journal=oncotarget&amp;page=article&amp;op=viewFile&amp;path%5B%5D=2480&amp;path%5B%5D=4418&amp;path%5B%5D=29158" alt="Identification of an erythroid progenitor-like signature and a prostate specific signature." width="500" height="284" title="Identification of an erythroid progenitor-like signature and a prostate specific signature." /&gt;&lt;/div&gt;        &lt;p class="FigureText"&gt;&lt;span class="FigureHeading"&gt;Figure 1: Identification of an erythroid progenitor-like signature and a prostate specific signature. &lt;/span&gt;The gene expression signature of single cells isolated from BM aspirates of patients with no evidence of disease [NED, pink along the top bar, 39&lt;/a&gt; cells; 4&lt;/a&gt; patients] and advanced patients [ADV, purple along the top bar, 46&lt;/a&gt; cells; 6&lt;/a&gt; patients] is shown in each column. Each color in the second horizontal bar represents a different patient. Cells are grouped according to EpCAM score indicated in the third horizontal bar. AHSP, CA1, HBA1, HBA2, HBB, HBD, LMO2, and MYB were used to identify an erythroid progenitor-like signature. The fourth horizontal bar denotes the erythroid progenitor-like score for each cell. AR, CD63, FOLH1, HOXB13, ID1, NKX3-1, RELB, and XAGE1A were used to identify a prostate specific signature. The fifth horizontal bar denotes the prostate specific signature intensity for each cell. Cells used or excluded from further analysis are depicted in blue or yellow, respectively, in the sixth horizontal bar.&lt;/p&gt;      &lt;/div&gt;  &lt;h3 class="SubHeading"&gt;Identification of an erythroid progenitor-like signature&lt;/h3&gt;       &lt;p class="BodyText"&gt;Upon examination of the gene expression array data of the 85 individual cells, there was a subset of cells with high expression of genes that are associated with erythroid progenitor-like cells [AHSP, CA1&lt;/a&gt;, HBA1&lt;/a&gt;, HBA2&lt;/a&gt;, HBB, HBD, LMO2&lt;/a&gt;, and MYB] [Figure 1&lt;/a&gt;]. Immature erythroid progenitor cells in the BM have been previously documented to transiently express EpCAM and E-cadherin [&lt;a href='#bib14' onclick="SelectText('bib14');return true;"&gt;14&lt;/a&gt;, &lt;a href='#bib15' onclick="SelectText('bib15');return true;"&gt;15&lt;/a&gt;]. To ensure the cells used for further analysis were prostate epithelial cells, we defined a dual gene expression signature for selection: EpCAM&lt;span class="BodySuper"&gt;+&lt;/span&gt;/CD45&lt;span class="BodySuper"&gt;-&lt;/span&gt; cells were defined as prostate epithelial cells if there was (i) low expression of erythroid progenitor-like signature including AHSP, CA1, HBA1, HBA2, HBB, HBD, LMO2, and MYB and (ii) high expression of prostate epithelial genes including AR, CD63, FOLH1, HOXB13, ID1, NKX3.1, RELB, and XAGE1A. NKX3.1 was the most consistent marker differentiating EpCAM+ prostate cells from EpCAM+ cells with the erythroid progenitor-like signature [Figure 1&lt;/a&gt;]. All of the cells fitting the two criteria of prostate epithelial cells had elevated expression of NKX3.1. Of the 85 cells originally isolated, 41 were excluded due to a high erythroid progenitor-like score (34/41 DTC) or a low prostate epithelial score including NKX3.1 (7/41 DTC) [Figure 1&lt;/a&gt;; Supplemental Figure 1&lt;/a&gt;]. Among the 34 cells that present a high erythroid progenitor-like score, 32 cells had an EpCAM intensity of 2&lt;span class="BodySuper"&gt;+ &lt;/span&gt;and were CD45&lt;span class="BodySuper"&gt;-&lt;/span&gt;. Only 2 cells that were CD45&lt;span class="BodySuper"&gt;- &lt;/span&gt;with an EpCAM intensity of 4&lt;span class="BodySuper"&gt;+&lt;/span&gt; had a high erythroid progenitor-like score. The remaining 44 cells, 7 cells from 4 patients in the NED group and 37 cells from 6 patients in the ADV group were considered PCa DTC and further analyzed [Supplemental Figure 1&lt;/a&gt;].&lt;/p&gt;       &lt;h3 class="SubHeading"&gt;DTC display heterogeneity within and between NED and ADV patients&lt;/h3&gt;       &lt;p class="BodyText"&gt;Cluster analysis of the 5000 most variable genes between the NED and ADV groups exhibited heterogeneity within DTC taken from patients in both the NED and ADV groups [Figure 2&lt;/a&gt;]. Not all DTC taken from patients in the NED group clustered together. DTC from NED and ADV patients were dispersed throughout the cluster. Additionally, not all DTC from the same patient cluster together. Of the 6 ADV patients included in the analysis, patient 2613_ADV showed 4 out of 5 DTC clustered together, however the remaining 5 ADV patients had DTC dispersed throughout the cluster [Figure 2&lt;/a&gt;]. &lt;/p&gt;      &lt;div class="OncoFigure"&gt;        &lt;div class="image"&gt;&lt;img src="https://www.oncotarget.com/index.php?journal=oncotarget&amp;page=article&amp;op=viewFile&amp;path%5B%5D=2480&amp;path%5B%5D=4418&amp;path%5B%5D=29159" alt="Cluster analysis of 5,000 most variable genes suggests heterogeneity between and within patient samples." width="550" height="477" title="Cluster analysis of 5,000 most variable genes suggests heterogeneity between and within patient samples." /&gt;&lt;/div&gt;        &lt;p class="FigureText"&gt;&lt;span class="FigureHeading"&gt;Figure 2: Cluster analysis of 5,000 most variable genes suggests heterogeneity between and within patient samples. &lt;/span&gt;An unsupervised cluster analysis of the 5,000 most variable genes from individual prostate cancer DTC isolated from the BM of patients with no evidence of disease [NED] (pink in the top bar, 2676, 2687, 2690, 2695) and advanced disease [ADV] (purple in the top bar, 2613, 2665, 2677, 2678, 2679, 2680). Each color in the second bar represents a different patient.&lt;/p&gt;  &lt;/div&gt;  &lt;h3 class="SubHeading"&gt;Identification of two gene signatures in DTC from ADV patients&lt;/h3&gt;       &lt;p class="BodyText"&gt;Clustering analyses based on the top 50 upregulated and 50 downregulated genes between the NED and ADV patient groups showed the DTC obtained from NED patients formed a single NED cluster [Figure 3&lt;/a&gt;]. DTC isolated from ADV patients formed two clusters (ADV_1 and ADV_2 clusters) in which ADV_1 had a gene expression profile that differed from that of the NED cluster, whereas ADV_2 had a gene expression profile that was similar to that of the NED cluster [Figure 4&lt;/a&gt;]. These results suggest that ADV patients harbor at least two distinct populations of DTC that may be associated with different ADV phenotypes (ADV_1 and ADV_2). Notably, all of the DTC isolated from patient 2613_ADV were in the ADV_2 cluster; all of the DTC isolated from patient 2665_ADV were in the ADV_1 cluster. The remaining 4 ADV patients had DTC in both the ADV_1 and ADV_2 clusters. Additionally, we determined that DTC EpCAM intensity levels do not distinguish between the ADV_1 and ADV_2 groups [Supplemental Figure 3&lt;/a&gt;].&lt;/p&gt;      &lt;div class="OncoFigure"&gt;        &lt;div class="image"&gt;&lt;img src="https://www.oncotarget.com/index.php?journal=oncotarget&amp;page=article&amp;op=viewFile&amp;path%5B%5D=2480&amp;path%5B%5D=4418&amp;path%5B%5D=29160" alt="Cluster analysis of the the top 50 upregulated and 50 downregulated genes segregates individual prostate cancer DTC into 3 categories." width="300" height="101" title="Cluster analysis of the the top 50 upregulated and 50 downregulated genes segregates individual prostate cancer DTC into 3 categories." /&gt;&lt;/div&gt;        &lt;p class="FigureText"&gt;&lt;span class="FigureHeading"&gt;Figure 3: Cluster analysis of the the top 50 upregulated and 50 downregulated genes segregates individual prostate cancer DTC into 3 categories. &lt;/span&gt;Hierarchical cluster analysis of individual prostate cancer DTC obtained from the BM based on the top 50 and bottom 50 genes differentially expressed between patients with no evidence of disease [NED] and patients with advanced disease [ADV]. Cells on the right of the figure were obtained from ADV patients (purple on the top horizontal bar) clustered into 2 groups. Cells on the left of the figure were obtained from NED patients (pink on the top horizontal bar) clustered into a single group. Each color in the second bar represents a different patient.&lt;/p&gt;      &lt;/div&gt;  &lt;h3 class="SubHeading"&gt;Components of the p38 pathway are differentially expressed in NED DTC and a subset of ADV DTC&lt;/h3&gt;       &lt;p class="BodyText"&gt;Gene set enrichment analysis identified pathways differentially regulated between the NED and ADV_1 clusters. In the ADV_1 cluster, genes involved in cytokine and chemokine signaling pathways were upregulated [p&amp;lt;0&lt;/a&gt;.001&lt;/a&gt;], whereas genes involved in the epithelial to mesenchymal transition and kinase inhibitor activity pathways were downregulated [p=0&lt;/a&gt;.013&lt;/a&gt;] and [p=0&lt;/a&gt;.007&lt;/a&gt;] respectively. Ingenuity Pathway Analysis (IPA) conducted on the 50 most upregulated genes and the 50 most downregulated genes showed that the p38 pathway was the top differentiating pathway involved between the NED and ADV_1 clusters [Figure 5&lt;/a&gt;]. We conducted a secondary analysis in an attempt to identify additional pathways altered between NED and ADV_1. We excluded the p38 pathway by looking at pathways that ranked second and third, both pathways involved ubiquitin C (UBC)-associated proteins. Furthermore, we used an alternate method by eliminating the twenty p38-associated genes and re-ran the IPA analysis. Again, UBC-related pathways showed up as the top two most significant pathways [Supplemental Figures 4&lt;/a&gt; and 5&lt;/a&gt;]. Based on these two analyses, the results suggest the p38 pathway is the dominant pathway that represents the top 50 and bottom 50 most differentially expressed genes. When we compared the NED vs. ADV and ADV_2 clusters, the top networks identified involved fibronectin, ERK1/2, and p38 [Supplemental Figure 6&lt;/a&gt; and 7&lt;/a&gt;].  Activation of the p38 pathway has been associated with DTC dormancy for solid tumors including HNSCC [&lt;a href='#bib16' onclick="SelectText('bib16');return true;"&gt;16&lt;/a&gt;], breast [&lt;a href='#bib17' onclick="SelectText('bib17');return true;"&gt;17&lt;/a&gt;], and prostate [&lt;a href='#bib18' onclick="SelectText('bib18');return true;"&gt;18&lt;/a&gt;]. Key genes, including KIAA1804 (MLK4&amp;#x3b2;), MTRR, and NUAK1 are downregulated in the NED cluster when compared with the ADV_1 cluster [Figure 4&lt;/a&gt;]. All three of these genes are involved in regulation of the p38 pathway [&lt;a href='#bib19' onclick="SelectText('bib19');return true;"&gt;19&lt;/a&gt;-&lt;a href='#bib21' onclick="SelectText('bib21');return true;"&gt;21&lt;/a&gt;]. It is important to note that not all p38-associated genes identified in the gene signature were present in the IPA analysis tool for IPA analysis. PIN4, a gene implicated in cell cycle re-entry, was also downregulated in the NED cluster [&lt;a href='#bib22' onclick="SelectText('bib22');return true;"&gt;22&lt;/a&gt;]. MALT1 and CDC25B, a positive regulator and an effector of the p38 pathway respectively [&lt;a href='#bib23' onclick="SelectText('bib23');return true;"&gt;23&lt;/a&gt;, &lt;a href='#bib24' onclick="SelectText('bib24');return true;"&gt;24&lt;/a&gt;], were upregulated in the NED cluster when compared with the ADV_1 cluster (Figure 4). Collectively, these data showed a distinct gene expression pattern associated with a potential dormancy-inducing p38 stress activated kinase pathway.&lt;/p&gt;      &lt;div class="OncoFigure"&gt;        &lt;div class="image"&gt;&lt;img src="https://www.oncotarget.com/index.php?journal=oncotarget&amp;page=article&amp;op=viewFile&amp;path%5B%5D=2480&amp;path%5B%5D=4418&amp;path%5B%5D=29161" alt="Heat map demonstrates individual prostate cancer DTC from patients with no evidence of disease [NED] are similar to a subset of prostate cancer DTC from advanced patients [ADV]." width="550" height="737" title="Heat map demonstrates individual prostate cancer DTC from patients with no evidence of disease [NED] are similar to a subset of prostate cancer DTC from advanced patients [ADV]." /&gt;&lt;/div&gt;        &lt;p class="FigureText"&gt;&lt;span class="FigureHeading"&gt;Figure 4: Heat map demonstrates individual prostate cancer DTC from patients with no evidence of disease [NED] are similar to a subset of prostate cancer DTC from advanced patients [ADV]. &lt;/span&gt;A supervised analysis of the top 50 and bottom 50 genes differentially expressed between NED patients and ADV_1 patients. Cells on the right of the figure were obtained from ADV patients and clustered into 2 groups (ADV_1: purple on top horizontal bar and ADV_2: pink on top horizontal bar). Cells on the left of the figure were obtained from NED patients and clustered into a single group (blue on top horizontal bar). Each color in the second bar represents a different patient.       &lt;/div&gt;        &lt;div class="OncoFigure"&gt;          &lt;div class="image"&gt;&lt;img src="https://www.oncotarget.com/index.php?journal=oncotarget&amp;page=article&amp;op=viewFile&amp;path%5B%5D=2480&amp;path%5B%5D=4418&amp;path%5B%5D=29162" alt="Genes associated with the p38 pathway are altered between no evidence of disease [NED] and a subset of advanced disease [ADV_1&amp;lt;/a&amp;gt;] prostate cancer DTC." width="550" height="392" title="Genes associated with the p38 pathway are altered between no evidence of disease [NED] and a subset of advanced disease [ADV_1&amp;lt;/a&amp;gt;] prostate cancer DTC." /&gt;&lt;/div&gt;          &lt;p class="FigureText"&gt;&lt;span class="FigureHeading"&gt;Figure 5: Genes associated with the p38 pathway are altered between no evidence of disease [NED] and a subset of advanced disease [ADV_1&lt;/a&gt;] prostate cancer DTC. &lt;/span&gt;Ingenuity Pathway Analysis identified the p38 stress response pathway as the top biological pathway altered in NED prostate cancer DTC vs. a subset of the advanced prostate cancer DTC [ADV_1&lt;/a&gt;]. Red indicates an increase in gene expression, green a decrease. Genes in uncolored nodes were not identified as differentially expressed in our microarray analyzes but were relevant to and therefore incorporated into individual networks based on the IPA database.&lt;/p&gt;  &lt;/div&gt;      &lt;/div&gt;  &lt;h3 class="SubHeading"&gt;DTC from patients with no evidence of clinical recurrence display a distinct dormancy signature&lt;/h3&gt;       &lt;p class="BodyText"&gt;The p38 stress response pathway activated by TGF&amp;#x3b2;2 and other cues has been proposed as a regulator of tumor cell dormancy [&lt;a href='#bib10' onclick="SelectText('bib10');return true;"&gt;10&lt;/a&gt;, &lt;a href='#bib16' onclick="SelectText('bib16');return true;"&gt;16&lt;/a&gt;, &lt;a href='#bib18' onclick="SelectText('bib18');return true;"&gt;18&lt;/a&gt;, &lt;a href='#bib25' onclick="SelectText('bib25');return true;"&gt;25&lt;/a&gt;, &lt;a href='#bib26' onclick="SelectText('bib26');return true;"&gt;26&lt;/a&gt;]. DTC from patients with NED were significantly enriched for p38-associated genes when compared to DTC from ADV patients. Our bioinformatics IPA analysis identified p38 stress response genes as being differentially regulated between the NED and ADV_1 DTC. Focusing on the top 50 upregulated and 50 downregulated genes, 15 p38-associated genes were upregulated in the top 50 upregulated genes and 6 p38-associated genes were significantly repressed in the top 50 downregulated genes in NED vs. ADV DTC. The 15 upregulated genes were significantly enriched in NED vs. ADV or the ADV_1 and ADV_2 groups (NED vs. ADV, p&amp;lt;0.0001; NED vs. ADV_1, p&amp;lt;0.0001; and NED vs. ADV_2, p&amp;lt;0.0001) [Figure 6&lt;/a&gt;A]. The six downregulated genes were significantly enriched in NED vs. ADV or the ADV_1 and ADV_2 groups (NED vs. ADV, p=0.0001; NED vs. ADV_1, p&amp;lt;0.001; and NED vs. ADV_2, p&amp;lt;0.001) [Figure 6&lt;/a&gt;B]. Interestingly, DTC in ADV_2 showed a significant enrichment of the dormancy upregulated and downregulated genes than the DTC in ADV_1 [Figure 6&lt;/a&gt;A and B]. We conclude that this analysis identified 21 genes (up- and downregulated) that were associated with p38 signaling and indolent behavior in NED DTC. We next tested whether a previously identified p38 dormancy signature was differentially enriched in NED and ADV DTC. We found that DTC from NED patients were significantly enriched with the 26 genes that were upregulated in a previously identified dormancy-associated signature (NED vs. ADV, p=0.01; NED vs. ADV_1, p=0.001; NED vs. ADV_2, p=0.002) [Figure 7&lt;/a&gt;A]. However, the p38-regulated signature did not distinguish between ADV_1 and ADV_2 groups [Figure 7&lt;/a&gt;A]. The 21 genes that were downregulated in the published dormancy-associated signature showed no significant difference between the NED vs. ADV_1, however statistical significance was reached when compared between NED vs ADV_2 (p=0.43) and NED vs ADV as a whole [p=0&lt;/a&gt;.03&lt;/a&gt;, Figure 7&lt;/a&gt;B]. Using two different approaches our data reveals that DTC from NED patients carry a p38-associated signature consistent with its role in induction of prolonged quiescence and DTC dormancy in experimental systems.&lt;/p&gt;      &lt;div class="OncoFigure"&gt;        &lt;div class="image"&gt;&lt;img src="https://www.oncotarget.com/index.php?journal=oncotarget&amp;page=article&amp;op=viewFile&amp;path%5B%5D=2480&amp;path%5B%5D=4418&amp;path%5B%5D=29163" alt="DTC analyses of p38 dormancy-associated genes." width="600" height="371" title="DTC analyses of p38 dormancy-associated genes." /&gt;&lt;/div&gt;        &lt;p class="FigureText"&gt;&lt;span class="FigureHeading"&gt;Figure 6: DTC analyses of p38 dormancy-associated genes. &lt;/span&gt;A. DTC from patients with no evidence of disease [NED] were significantly enriched when compared to DTC from advanced patients [ADV] with 14 p38-associated genes that were upregulated in the top 50 differentially expressed genes in NED vs. ADV, and two subsets of advanced patient [ADV_1&lt;/a&gt;, and ADV_2&lt;/a&gt;] DTC. B. The 6 genes that were downregulated in NED DTC were significantly repressed vs. ADV, ADV_1, and ADV_2 DTC. Mean&amp;plusmn;SD is shown. NED was compared to ADV, ADV_1, or ADV_2, and p&amp;lt;0.05 was considered statistically significant.        &lt;/div&gt;        &lt;div class="OncoFigure"&gt;          &lt;div class="image"&gt;&lt;img src="https://www.oncotarget.com/index.php?journal=oncotarget&amp;page=article&amp;op=viewFile&amp;path%5B%5D=2480&amp;path%5B%5D=4418&amp;path%5B%5D=29164" alt="DTC analyses of dormancy-associated genes." width="550" height="317" title="DTC analyses of dormancy-associated genes." /&gt;&lt;/div&gt;          &lt;p class="FigureText"&gt;&lt;span class="FigureHeading"&gt;Figure 7: DTC analyses of dormancy-associated genes. &lt;/span&gt;A. A dormancy-related signature identified through previous dormancy studies (12-15) was used to compare to the signature obtained from the gene expression arrays. A. DTC from patients with no evidence of disease [NED] were significantly enriched for 26 dormancy-related genes that are upregulated in NED vs. DTC in advanced patients [ADV], and two subsets of advanced patient [ADV_1&lt;/a&gt;, and ADV_2&lt;/a&gt;] DTC. B. Twenty-one genes that were downregulated in a dormancy-related signature were not repressed in NED vs. ADV or when compared to ADV_1 or ADV_2 DTC. Mean&amp;plusmn;SD is shown. NED was compared to ADV, ADV_1, or ADV_2, and p&amp;lt;0.05 was considered statistically significant.&lt;/p&gt;        &lt;/div&gt;      &lt;/div&gt;  &lt;h2 class="Heading"&gt;Discussion&lt;/h2&gt;       &lt;p class="BodyText"&gt;We identified and characterized a definitive set of DTC in patients with a high prostate epithelial score and a low erythroid progenitor-like score. The majority of the PCa DTC had an EpCAM score of 3+ or 4+, with only a few cells displaying an EpCAM score of 2+. We profiled individual cells instead of pools of cells to avoid the inclusion of any contaminating cell types in our signatures, which would confound data analyses for definitive PCa DTC. Until further evidence is available to suggest that the predominantly EpCAM 2+ cells with a low prostate epithelial score and a high erythroid-progenitor-like score are associated with PCa, we determined that based on the extensive cellular plasticity present in these cells that they should be removed from our analysis. Of note, it is unlikely that these erythroid progenitor-like cells are present in the blood contaminating the analysis of CTC as erythropoiesis usually occurs in the BM, however the possibility should not be ruled out, e.g. extrameduallary erythropoiesis can occur in myelofibrosis [&lt;a href='#bib27' onclick="SelectText('bib27');return true;"&gt;27&lt;/a&gt;].&lt;/p&gt;       &lt;p class="BodyText"&gt;DTC from most patients (except one patient) unsurprisingly were heterogeneous between and within patient samples at the transcriptional level, regardless of disease type (i.e. NED or ADV) [&lt;a href='#bib28' onclick="SelectText('bib28');return true;"&gt;28&lt;/a&gt;, &lt;a href='#bib29' onclick="SelectText('bib29');return true;"&gt;29&lt;/a&gt;]. One could speculate that DTC obtained from one patient may all arise from a single micrometastasis (e.g. patient 2613), providing some clonality and a similar microenvironment for each of these cells. However, in the other patients, cells may have been isolated from different micrometastases or individual cells present in the BM and this would explain why some but not all cells from patients cluster together. Similar reasoning/hypothesis may explain why ADV_2 clusters with NED, with ADV_2 and NED cells being present as individual cells and ADV_1 cells being isolated from groups of cells.&lt;span class="BodyItalics"&gt; &lt;/span&gt;Of note, when we clustered individual DTC based on the expression of the top and bottom 50 differentially expressed genes in NED and ADV_1, we observed a greater degree of patient-associated clustering of the individual DTC [Figure 4&lt;/a&gt;]. This suggests the level of heterogeneity observed will depend on the number of genes, or more specifically the biological process assessed in the signature. &lt;/p&gt;       &lt;p class="BodyText"&gt;DTC in ADV patients could be clustered into two groups: ADV_1 and ADV_2. Of note, we observed no difference between biochemical and radiographic disease states in ADV patient samples when compared to NED patient samples (data not shown). Among the patients in the ADV category, 4 of 6 had DTC in both the ADV_1 and ADV_2 clusters. This indicates that the DTC in the patients with clinically advanced disease harbor cells in different biological states. For the top 50 and bottom 50 genes differentially expressed between NED and ADV_1 categories, surprisingly the ADV_2 DTC were similar to the NED DTC. This does not necessarily mean the ADV_2 DTC were also in a state of dormancy, but the similarity in the gene expression pattern suggests potential concordance in their behavior. This is of clinical importance, as the current therapies for treatment of advanced PCa targets cells that are actively replicating. Cells in a state of dormancy/quiescence would not be affected by these treatment strategies, and they could potentially exit dormancy and become active metastases after treatment has concluded. If the ADV_2 DTC possess some of the hallmarks of the DTC from the NED patients, it could be hypothesized that a subpopulation of PCa cells in the BM of the ADV patients are in a dormant/quiescent state. Further, the challenge for cure in these patients would rely on treating both the active and dormant PCa DTC. &lt;/p&gt;       &lt;p class="BodyText"&gt;The identification of clinically-relevant pathways involved in PCa dormancy in patients is needed to develop dormancy models, identify robust markers and design effective treatment strategies. The activation of the p38 pathway has been identified to promote and maintain tumor dormancy [&lt;a href='#bib10' onclick="SelectText('bib10');return true;"&gt;10&lt;/a&gt;, &lt;a href='#bib16' onclick="SelectText('bib16');return true;"&gt;16&lt;/a&gt;, &lt;a href='#bib18' onclick="SelectText('bib18');return true;"&gt;18&lt;/a&gt;, &lt;a href='#bib25' onclick="SelectText('bib25');return true;"&gt;25&lt;/a&gt;, &lt;a href='#bib26' onclick="SelectText('bib26');return true;"&gt;26&lt;/a&gt;]. A recent report demonstrated that PCa stem-like cells in the BM enter a p38-dependent dormancy state in an animal model, however the clinical relevance of the p38 pathway in PCa dormancy is yet to be established [&lt;a href='#bib18' onclick="SelectText('bib18');return true;"&gt;18&lt;/a&gt;]. In our study, genes associated with the p38 stress response pathway were shown to be altered when we compared DTC from NED patients to a group of DTC from ADV patients. In addition to the genes identified in the IPA analysis, both KIAA1804 (MLK4&amp;#x3b2;) and MTRR were downregulated in the NED DTC [Supplemental Table &lt;a href='#bib2' onclick="SelectText('bib2');return true;"&gt;2&lt;/a&gt;]. The mitogen-activated protein kinase family and specifically MLK4&amp;#x3b2; has been shown to inhibit the p38 pathway [&lt;a href='#bib19' onclick="SelectText('bib19');return true;"&gt;19&lt;/a&gt;]. MTRR is an electron transferase involved in methionine synthesis and has also been shown to inhibit the p38 pathway [&lt;a href='#bib21' onclick="SelectText('bib21');return true;"&gt;21&lt;/a&gt;]. Thus, loss of these genes may be more permissive for persistent p38 activation in DTC in PCa patients. &lt;/p&gt;       &lt;p class="BodyText"&gt;NUAK1 is a protein kinase that has been linked to the activation of the p38 pathway and cell proliferation [&lt;a href='#bib20' onclick="SelectText('bib20');return true;"&gt;20&lt;/a&gt;]. NUAK1 was found to be upregulated in the ADV_1 DTC population. PIN4 was also upregulated in the ADV_1 DTC group compared to the NED DTC. PIN4 function is linked to cell proliferation [&lt;a href='#bib30' onclick="SelectText('bib30');return true;"&gt;30&lt;/a&gt;]. MALT1, which showed upregulation in the NED DTC, is a cysteine protease whose function has been linked to activation of the p38 pathway [&lt;a href='#bib23' onclick="SelectText('bib23');return true;"&gt;23&lt;/a&gt;]. MALT1 regulates p38 and CDC25B is a downstream target of p38 [&lt;a href='#bib24' onclick="SelectText('bib24');return true;"&gt;24&lt;/a&gt;]. CDC25B, a phosphatase that activates the cyclin-dependent kinase CDC2, was upregulated in NED DTC. p38 phosphorylates CDC25B, which results in an inhibition of entry into the M phase of the cell cycle [&lt;a href='#bib24' onclick="SelectText('bib24');return true;"&gt;24&lt;/a&gt;]. Elevated mRNA levels for CDC25B might be a compensatory mechanism for p38-induced growth arrest. While extrapolating from gene expression to function is not completely reliable, the elevated expression of these 6 genes discussed above suggests a role for the p38 stress-response pathway in the DTC isolated from the BM of the NED patients.&lt;/p&gt;       &lt;p class="BodyText"&gt;We developed a p38-associated dormancy signature based on the top 50 and bottom 50 differentially expressed genes between NED DTC and ADV_1 DTC. This signature, while biased, could clearly differentiate between NED DTC and ADV DTC. To further validate our findings we interrogated the DTC expression profiles for a dormancy signature previously developed from a HNSCC model [&lt;a href='#bib16' onclick="SelectText('bib16');return true;"&gt;16&lt;/a&gt;, &lt;a href='#bib25' onclick="SelectText('bib25');return true;"&gt;25&lt;/a&gt;, &lt;a href='#bib31' onclick="SelectText('bib31');return true;"&gt;31&lt;/a&gt;] and a PCa model [&lt;a href='#bib18' onclick="SelectText('bib18');return true;"&gt;18&lt;/a&gt;]. Dormancy upregulated genes were enriched in the NED DTC, suggesting that the dormancy signature could consistently identify dormant PCa cells present in the patient BM. Among the dormancy-up genes in NED DTC, TGF&amp;#x3b2;2 and BMP7 have been implicated specifically in dormancy of DTC in the BM [&lt;a href='#bib16' onclick="SelectText('bib16');return true;"&gt;16&lt;/a&gt;, &lt;a href='#bib18' onclick="SelectText('bib18');return true;"&gt;18&lt;/a&gt;]. However, the dormancy downregulated genes could not distinguish between the NED, ADV_1 and ADV_2 DTC. One possible explanation for this is that solitary PCa DTC may be slow cycling or non-proliferative in nature, therefore the dormancy-down genes that are linked to proliferation may not be changing sufficiently. Furthermore, dormancy initiation and maintenance may require an induction of the p38 stress response pathway, which results in the activation of different downstream cell cycle effectors among different tumor types.&lt;/p&gt;       &lt;p class="BodyText"&gt;The lack of clinically-relevant models in the dormancy field is a challenge that needs to be overcome. New model systems involving TGF-beta and the perivascular niche are emerging and may be relevant for PCa DTC study if proven clinically relevant [&lt;a href='#bib16' onclick="SelectText('bib16');return true;"&gt;16&lt;/a&gt;, &lt;a href='#bib32' onclick="SelectText('bib32');return true;"&gt;32&lt;/a&gt;]. Despite the clinical relevance, our study is restricted in its interpretation due to the limited number of EpCAM+ cells that can be isolated from NED patient BM and the fact that we were unable to further interrogate individual cells at the protein level or functionally to validate our findings. Nonetheless, our study provides the first clinical evidence supporting a role for the p38 pathway in dormancy in PCa DTC in patients, which supports further functional validation &lt;span class="BodyItalics"&gt;in vitro&lt;/span&gt; and &lt;span class="BodyItalics"&gt;in vivo&lt;/span&gt;. &lt;/p&gt;       &lt;p class="BodyText"&gt;In summary, this study illustrates a first-in-field transcriptomic signature from individual DTC in cancer patients. We established a gene signature to definitively identify PCa DTC from other EpCAM&lt;span class="BodySuper"&gt;+&lt;/span&gt;/CD45&lt;span class="BodySuper"&gt;-&lt;/span&gt; cells that do not demonstrate typical prostate epithelial markers in the BM. We further demonstrated that patients with advanced disease may harbor at least two populations of DTC, in which one group of DTC is similar to those isolated from patients with NED for up to 18 years. We also validated our signature using a mechanistically modeled dormancy-associated signature from head and neck cancer supporting the hypothesis that the p38 pathway is associated with the dormant nature of these DTC in NED patients. Ultimately, in the clinical setting, the intent is to maintain DTC in a dormant state or to better understand dormancy such that active disseminated cells can be shifted to a state of dormancy, and hence delay the onset of symptomatic and lethal metastatic disease. Our results suggest that the p38 pathway may serve as a potential biomarker to monitor early recurrence and identifies the p38 axis as an attractive target for therapeutic intervention to regulate dormancy in PCa patients. &lt;/p&gt;       &lt;h2 class="Heading"&gt;Methods&lt;/h2&gt;       &lt;h3 class="SubHeading"&gt;Selection criteria for patients&lt;/h3&gt;       &lt;p class="BodyText"&gt;Patients from 2 clinical categories were selected. Patients who had a diagnosis of adenocarcinoma of the prostate, had undergone a radical prostatectomy (RP), had not received neoadjuvant or adjuvant treatment and had a PSA that was undetectable (less than 0.1 ng/mL) for a minimum of 7 years (range 7-18 years) after RP, were considered to have no evidence of disease (NED). Patients with advanced disease (ADV) were defined as either having radiographic evidence of metastatic disease at the time of diagnosis of adenocarcinoma of the prostate or having biochemical recurrence (defined as two consecutive rises in PSA &amp;gt; 0.2 ng/mL in a patient who previously had a RP and an undetectable PSA), after primary curative therapies for adenocarcinoma of the prostate. &lt;/p&gt;       &lt;h3 class="SubHeading"&gt;Enrichment of DTC from the BM of PCa patients &lt;/h3&gt;       &lt;p class="BodyText"&gt;All materials were acquired and used conforming to IRB-approved protocols at the University of Washington. DTC were isolated from BM samples of PCa patients as previously reported [&lt;a href='#bib13' onclick="SelectText('bib13');return true;"&gt;13&lt;/a&gt;]. Briefly, ten ml of BM was aspirated from the posterior iliac crest into a 30 ml syringe containing 10 ml of 6% sodium citrate under local anesthesia. Processing of samples commenced within 1&amp;#x2013;2 hours and was completed within 5 hours. BM aspirates were separated using 15 ml of Ficoll-Isopaque (1.077 g/ml) (Accurate Chemical, Westbury, NY). Centrifugation yielded a mononuclear cell layer containing DTC, which underwent immunomagnetic selection with the MACS system (Miltenyi Biotec, Auburn, CA). Anti-CD45 and anti-CD61 antibodies were used first for negative selection to eliminate leukocytes, megakaryocytes, and platelets. Positive selection was then performed with immunomagnetic beads coated with anti-human epithelial antigen (HEA) antibodies to target epithelial cells. &lt;/p&gt;       &lt;h3 class="SubHeading"&gt;Isolation of individual DTC&lt;/h3&gt;       &lt;p class="BodyText"&gt;For identification and isolation of DTC, the enriched population was subjected to immunostaining with fluorescein isothiocyanate labeled anti-BerEP4/EpCAM antibodies (Dako, Carpinteria, CA), which bind a different epitope on HEA than the anti-HEA antibody used for positive selection. A phycoerythrin conjugated anti-CD45 antibody was also added for identification of leukocytes. EpCAM&lt;span class="BodySuper"&gt;+&lt;/span&gt;/CD45&lt;span class="BodySuper"&gt;-&lt;/span&gt; cells were kept on ice, viewed under fluorescent light, and intact cells with a clearly defined EpCAM labelled membrane were isolated using a micromanipulator. Based on historic testing of this method using the trypan blue exclusion assay, DTC picked at this stage of isolation are viable. Individual cells were lysed with 2 &amp;micro;l of WT-Ovation&amp;#x2122; One-Direct Amplification System (NuGEN) lysis buffer and stored at &amp;#x2212;80&lt;span class="BodySuper"&gt;o&lt;/span&gt;C for gene expression profiling.&lt;/p&gt;       &lt;h3 class="SubHeading"&gt;Amplification of total RNA from individual DTC&lt;/h3&gt;       &lt;p class="BodyText"&gt;Total RNA was amplified from each sample using the WT-Ovation&amp;#x2122; One-Direct Amplification System (NuGEN) according to the manufacturer&amp;#x2019;s directions. The use of an aluminum cooling block on ice facilitated the handling of the reaction tubes. The amplified cDNA products passed the purity test of showing Abs&lt;span class="BodySub"&gt;260&lt;/span&gt;/Abs&lt;span class="BodySub"&gt;280&lt;/span&gt; ratios between 1.9 to 2.0, and a subset was analyzed on an Agilent Bioanalyzer using the RNA 6000 Pico LabChip with the mRNA Pico program to assess size distribution as previously described [&lt;a href='#bib13' onclick="SelectText('bib13');return true;"&gt;13&lt;/a&gt;]. Post-SPIA modification and post-amplification work was performed in a separate workspace. &lt;/p&gt;       &lt;h3 class="SubHeading"&gt;Labeling and hybridization of amplified material on Agilent chips&lt;/h3&gt;       &lt;p class="BodyText"&gt;Amplified cDNA from each sample were labeled using the BioPrime&amp;reg; Total Genomic Labeling System (Invitrogen&amp;#x2122;). Hybridization probes were prepared by combining 6 mg of Alexa Fluor&amp;reg; 3 labeled sample and 1 mg Alexa Fluor&amp;reg; 5 labeled reference and denatured at 95&amp;deg;C and hybridized at 63&amp;deg;C on Agilent Human 4x44K microarrays and processed following the manufacturer&amp;#x2019;s specifications. Arrays were scanned on an Agilent DNA Microarray Scanner [&lt;a href='#bib13' onclick="SelectText('bib13');return true;"&gt;13&lt;/a&gt;].&lt;/p&gt;       &lt;h3 class="SubHeading"&gt;DTC Selection&lt;/h3&gt;       &lt;p class="BodyText"&gt;We analyzed DTC from 5/9 NED patients. The remaining 4 patients all had DTC, but had too few to move forward and analyze. These samples were not processed as our goal was to obtain approximately 10 cells/patient to analyze tumor cell heterogeneity, whereas these patients only had 1-4 cells. We analyzed DTC from 6/8 ADV sampled patients. The DTC isolated from one patient were of poor quality, the other patient had no DTC observed in the bone marrow sample. &lt;/p&gt;       &lt;h3 class="SubHeading"&gt;Gene expression analysis&lt;/h3&gt;       &lt;p class="BodyText"&gt;Data were loess normalized within arrays and quantile normalized among arrays in R using the Limma Bioconductor package. Samples with poor microarray hybridization signals were removed. Data were filtered to remove probes with mean signal intensities below 300. The Statistical Analysis of Microarray (SAM) program (http://www-stat.stanford.edu/~tibs/SAM/) was used to analyze expression differences between groups. Unpaired, two-sample t-tests were calculated for 11202 unique genes passing filters and controlled for multiple testing by estimation of q-values using the false discovery rate (FDR) method. Microarray data are deposited in the Gene Expression Omnibus database under the accession number GSE48995. To determine whether transcriptomic differences observed in ADV versus NED groups were enriched for genes within canonical pathways and Gene Ontology gene sets, the t-test results were subjected to Gene Set Enrichment Analysis (GSEA) using preranked mode with permutation testing of the gene sets to adjust for multiple hypothesis testing, generating an FDR. Global unsupervised, hierarchical clustering analysis was performed with the top 5000 most variable genes among all samples as determined by inter-quartile range (IQR). Unsupervised, hierarchical clustering of the top and bottom 50 most differentially expressed between NED and ADV groups based on SAM score identified subgroups ADV_1 and ADV_2. Erythroid progenitor-like and prostate enrichment scores were calculated using the Gene Set Variation Analysis (GSVA) Bioconductor package in R [&lt;a href='#bib33' onclick="SelectText('bib33');return true;"&gt;33&lt;/a&gt;].&lt;/p&gt;       &lt;h3 class="SubHeading"&gt;Ingenuity Pathway Analysis&lt;/h3&gt;       &lt;p class="BodyText"&gt;The top and bottom 50 most differentially expressed genes (total 100 genes) between NED, ADV, ADV_1 and ADV_2 groups were imported into Ingenuity Pathway Analysis (IPA, Ingenuity Systems; https://www.ingenuity.com) to identify biological pathways involved in different groups of DTC. IPA is a repository of biological interactions and functional annotations to demonstrate relationships between proteins, genes, complexes, metabolites, and drugs. The IPA knowledge library includes information from NCBI databases (EntrezGene, RefSeq, OMIM disease associations), microRNA-mRNA target databases, GWAS (Genome Wide Analysis Study) databases, and Kyoto Encyclopedia of Genes and Genomes (KEGG).&lt;/p&gt;       &lt;h3 class="SubHeading"&gt;Analysis of DTC expression profiles for the presence of a p38&amp;#x3b1;/&amp;#x3b2;-regulated dormancy signature&lt;/h3&gt;       &lt;p class="BodyText"&gt;For the p38&amp;#x3b1;/&amp;#x3b2; dormancy-associated signature identified in the IPA, the enrichment for dormancy upregulated or downregulated genes for each DTC was calculated. Specifically, the percent of dormancy upregulated genes that were also upregulated in the DTC was scored as the percent coverage of dormancy UP genes induced for each individual DTC. The same was applied to the downregulated genes in the above-mentioned signature. Cutoffs were the same as those used for all normalization analysis. In all cases, differences in means were estimated using a linear regression model and statistical significance was evaluated using t-tests of appropriate model coefficients. For the previously identified tumor dormancy-associated signature, expression profiles of the NED, ADV, ADV_1 or ADV_2 subgroups of DTC were parsed for the genes upregulated and downregulated in a previously published tumor dormancy gene signature of 19 upregulated and 21 downregulated genes linked to p38&amp;#x3b1;/&amp;#x3b2; activation in a head and neck squamous cell carcinoma (HNSCC) model [&lt;a href='#bib16' onclick="SelectText('bib16');return true;"&gt;16&lt;/a&gt;, &lt;a href='#bib25' onclick="SelectText('bib25');return true;"&gt;25&lt;/a&gt;, &lt;a href='#bib31' onclick="SelectText('bib31');return true;"&gt;31&lt;/a&gt;] or to BMP7 signaling in a PCa model [&lt;a href='#bib18' onclick="SelectText('bib18');return true;"&gt;18&lt;/a&gt;]. The latter signature only added BMP7. This signature was expanded to incorporate an additional 7 genes that have been linked to &lt;span class="BodyItalics"&gt;in vivo&lt;/span&gt; quiescence of dormant HNSCC cells (Aguirre-Ghiso et al., unpublished). The percent of upregulated genes in the signatures that were also upregulated in the DTC was scored as the percent coverage of dormancy UP genes induced for each individual DTC. The same was applied to the downregulated genes in the above-mentioned signatures. Cutoffs were the same used for all normalization analysis. For example, when 26 of the genes upregulated in the expanded signature (19+7 genes induced in the HNSCC and PCa model) were all upregulated in an individual DTC, that DTC was scored as having 100% of coverage of the dormancy UP genes. The same was applied for downregulated genes. In all cases, differences in means were estimated using a linear regression model and statistical significance was evaluated using t-tests of appropriate model coefficients.&lt;/p&gt;       &lt;h2 class="Heading"&gt;Acknowledgements&lt;/h2&gt;       &lt;p class="BodyText"&gt;We would like to thank the patients who donated BM aspirates that made this work possible. These studies were supported by resources from NIH RC1 CA144825 ARRA Challenge, Janssen Research and Development LLC, NIH PO1 CA85859, U01 CA164188, the PNW Prostate Cancer SPORE NIH P50 CA097186 to P.S.N., and Samuel Waxman Cancer Research Foundation Tumor Dormancy Program NIH CA109182 and NIH CA163131 to J.A.A-G. H.M.L. is a recipient of a Young Investigator Award from Prostate Cancer Foundation and a Career Development Award from NIH Pacific Northwest Prostate Cancer SPORE. This material is also the result of work supported by resources from the VA Puget Sound Health Care System, Seattle, Washington (R.L.V is a research career scientist, P.H.L is a staff physician).&lt;/p&gt;       &lt;h2 class="Heading"&gt;References&lt;/h2&gt;       &lt;p class="ReferenceList" id="bib1"&gt;1. Siegel R, Ma J, Zou Z and Jemal A. Cancer statistics, 2014. CA Cancer J Clin. 2014; 64(1):9-29.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib2"&gt;2. Cooperberg MR, Broering JM and Carroll PR. Time trends and local variation in primary treatment of localized prostate cancer. J Clin Oncol. 2010; 28(7):1117-23.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib3"&gt;3. Morton RA, Steiner MS and Walsh PC. Cancer control following anatomical radical prostatectomy: an interim report. J Urol. 1991; 145(6):1197-200.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib4"&gt;4. Cheng L, Zincke H, Blute ML, Bergstralh EJ, Scherer B and Bostwick DG. Risk of prostate carcinoma death in patients with lymph node metastasis. Cancer. 2001; 91(1):66-73.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib5"&gt;5. Weiss L. Random and nonrandom processes in metastasis, and metastatic inefficiency. Invasion Metastasis. 1983; 3(4):193-207.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib6"&gt;6. Fidler IJ. Metastasis: guantitative analysis of distribution and fate of tumor embolilabeled with 125 I-5-iodo-2&amp;#x2019;-deoxyuridine. J Natl Cancer Inst. 1970; 45(4):773-82.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib7"&gt;7. Butler TP, Gullino PM. Quantitation of cell shedding into efferent blood of mammary adenocarcinoma. Cancer Res. 1975; 35(3):512-6.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib8"&gt;8. Chambers AF, Groom AC and MacDonald IC. Dissemination and growth of cancer cells in metastatic sites. Nat Rev Cancer. 2002; 2(8):563-72.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib9"&gt;9. Hadfield G. Dormancy in Cancer. Br Med J. 1954; 2(4888):635.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib10"&gt;10. Aguirre-Ghiso JA, Estrada Y, Liu D and Ossowski L. ERK(MAPK) activity as a determinant of tumor growth and dormancy; regulation by p38(SAPK). Cancer Res. 2003; 63(7):1684-95.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib11"&gt;11. Ruppender NS, Morrissey C, Lange PH and Vessella RL. Dormancy in solid tumors: implications for prostate cancer. Cancer Metastasis Rev. 2013; 32(3-4):501-9.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib12"&gt;12. Ahove DA, Hoffman KE, Hu JC, Choueiri TK, D&amp;#x2019;Amico AV and Nguyen PL. Which patients with undetectable PSA levels 5 years after radical prostatectomy are still at risk of recurrence?--implications for a risk-adapted follow-up strategy. Urology. 2010; 76(5):1201-5.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib13"&gt;13. Welty CJ, Coleman I, Coleman R, Lakely B, Xia J, Chen S, Gulati R, Larson SR, Lange PH, Montgomery B, Nelson PS, Vessella RL and Morrissey C. Single cell transcriptomic analysis of prostate cancer cells. BMC Mol Biol. 2013; 146.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib14"&gt;14. Lammers R, Giesert C, Grunebach F, Marxer A, Vogel W and Buhring HJ. Monoclonal antibody 9C4 recognizes epithelial cellular adhesion molecule, a cell surface antigen expressed in early steps of erythropoiesis. Exp Hematol. 2002; 30(6):537-45.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib15"&gt;15. Eisenwort G, Jurkin J, Yasmin N, Bauer T, Gesslbauer B and Strobl H. Identification of TROP2 (TACSTD2), an EpCAM-like molecule, as a specific marker for TGF-beta1-dependent human epidermal Langerhans cells. J Invest Dermatol. 2011; 131(10):2049-57.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib16"&gt;16. Bragado P, Estrada Y, Parikh F, Krause S, Capobianco C, Farina HG, Schewe DM and Aguirre-Ghiso JA. TGF-beta2 dictates disseminated tumour cell fate in target organs through TGF-beta-RIII and p38alpha/beta signalling. Nat Cell Biol. 2013; 15(11):1351-61.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib17"&gt;17. Marshall JC, Collins JW, Nakayama J, Horak CE, Liewehr DJ, Steinberg SM, Albaugh M, Vidal-Vanaclocha F, Palmieri D, Barbier M, Murone M and Steeg PS. Effect of inhibition of the lysophosphatidic acid receptor 1 on metastasis and metastatic dormancy in breast cancer. J Natl Cancer Inst. 2012; 104(17):1306-19.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib18"&gt;18. Kobayashi A, Okuda H, Xing F, Pandey PR, Watabe M, Hirota S, Pai SK, Liu W, Fukuda K, Chambers C, Wilber A and Watabe K. Bone morphogenetic protein 7 in dormancy and metastasis of prostate cancer stem-like cells in bone. J Exp Med. 2011; 208(13):2641-55.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib19"&gt;19. Abi Saab WF, Brown MS and Chadee DN. MLK4beta functions as a negative regulator of MAPK signaling and cell invasion. Oncogenesis. 2012; 1e6.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib20"&gt;20. Ratkaj I, Bujak M, Jurisic D, Baus LM, Bendelja K, Pavelic K and Kraljevic PS. Microarray analysis of Dupuytren&amp;#x2019;s disease cells: the profibrogenic role of the TGF-beta inducible p38 MAPK pathway. Cell Physiol Biochem. 2012; 30(4):927-42.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib21"&gt;21. Richard E, Desviat LR, Ugarte M and Perez B. Oxidative stress and apoptosis in homocystinuria patients with genetic remethylation defects. J Cell Biochem. 2013; 114(1):183-91.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib22"&gt;22. Mueller JW, Bayer P. Small family with key contacts: par14 and par17 parvulin proteins, relatives of pin1, now emerge in biomedical research. Perspect Medicin Chem. 2008; 211-20.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib23"&gt;23. Ruland J, Duncan GS, Wakeham A and Mak TW. Differential requirement for Malt1 in T and B cell antigen receptor signaling. Immunity. 2003; 19(5):749-58.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib24"&gt;24. Cha H, Wang X, Li H and Fornace AJ, Jr. A functional role for p38 MAPK in modulating mitotic transit in the absence of stress. J Biol Chem. 2007; 282(31):22984-92.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib25"&gt;25. Adam AP, George A, Schewe D, Bragado P, Iglesias BV, Ranganathan AC, Kourtidis A, Conklin DS and Aguirre-Ghiso JA. Computational identification of a p38SAPK-regulated transcription factor network required for tumor cell quiescence. Cancer Res. 2009; 69(14):5664-72.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib26"&gt;26. Aguirre-Ghiso JA, Liu D, Mignatti A, Kovalski K and Ossowski L. Urokinase receptor and fibronectin regulate the ERK(MAPK) to p38(MAPK) activity ratios that determine carcinoma cell proliferation or dormancy &lt;span class="BodyItalics"&gt;in vivo&lt;/span&gt;. Mol Biol Cell. 2001; 12(4):863-79.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib27"&gt;27. Sjogren U. Different composition of the erythropoietic tissue in bone marrow, spleen and liver in myelofibrosis. Acta Haematol. 1978; 59(4):231-6.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib28"&gt;28. Morrissey C, Roudier MP, Dowell A, True LD, Ketchanji M, Welty C, Corey E, Lange PH, Higano CS and Vessella RL. Effects of androgen deprivation therapy and bisphosphonate treatment on bone in patients with metastatic castration-resistant prostate cancer: results from the University of Washington Rapid Autopsy Series. J Bone Miner Res. 2013; 28(2):333-40.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib29"&gt;29. Roudier MP, Morrissey C, True LD, Higano CS, Vessella RL and Ott SM. Histopathological assessment of prostate cancer bone osteoblastic metastases. J Urol. 2008; 180(3):1154-60.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib30"&gt;30. Uchida T, Takamiya M, Takahashi M, Miyashita H, Ikeda H, Terada T, Matsuo Y, Shirouzu M, Yokoyama S, Fujimori F and Hunter T. Pin1 and Par14 peptidyl prolyl isomerase inhibitors block cell proliferation. Chem Biol. 2003; 10(1):15-24.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib31"&gt;31. Ranganathan AC, Zhang L, Adam AP and Aguirre-Ghiso JA. Functional coupling of p38-induced up-regulation of BiP and activation of RNA-dependent protein kinase-like endoplasmic reticulum kinase to drug resistance of dormant carcinoma cells. Cancer Res. 2006; 66(3):1702-11.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib32"&gt;32. Ghajar CM, Peinado H, Mori H, Matei IR, Evason KJ, Brazier H, Almeida D, Koller A, Hajjar KA, Stainier DY, Chen EI, Lyden D and Bissell MJ. The perivascular niche regulates breast tumour dormancy. Nat Cell Biol. 2013; 15(7):807-17.&lt;/p&gt;       &lt;p class="ReferenceList" id="bib33"&gt;33. Hanzelmann S, Castelo R and Guinney J. GSVA: gene set variation analysis for microarray and RNA-seq data. BMC Bioinformatics. 2013; 147.&lt;/p&gt;      &lt;/div&gt;    &lt;/div&gt;      </code></p><p><code>568</code><code>&lt;!-- end &lt;/html&gt; --&gt;</code></p><p><code>569</code><code>          </code></p><p><code>570</code><code>	</code></p><p><code>571</code><code>          </code></p><p><code>572</code><code>          </code></p><p><code>573</code><code>&lt;script type="text/javascript"&gt;</code></p><p><code>574</code><code>	if ($("#extendedInfo").length || $("#PDFer").length) {</code></p><p><code>575</code><code>		$(".oldStyle").hide();</code></p><p><code>576</code><code>		$("#sidebarRTAuthorBios").hide();</code></p><p><code>577</code><code>	}</code></p><p><code>578</code><code>	if(!$('#AuthorInformation').length) {</code></p><p><code>579</code><code>		$(".oldStyle").hide();</code></p><p><code>580</code><code>		$("#sidebarRTAuthorBios").hide();</code></p><p><code>581</code><code>	}</code></p><p><code>582</code><code>&lt;/script&gt;</code></p><p><code>583</code><code>          </code></p><p><code>584</code><code>          </code></p><p><code>585</code><code>          </code></p><p><code>586</code><code>          </code></p><p><code>587</code><code>&lt;script type="text/javascript"&gt;</code></p><p><code>588</code><code>var rmHeaderBorderTop = {</code></p><p><code>589</code><code>	noBD: function(){</code></p><p><code>590</code><code>		var artPageHeadings = document.getElementsByClassName('Heading');</code></p><p><code>591</code><code>		for (var i = 0; i &lt; artPageHeadings.length; i++) {</code></p><p><code>592</code><code>			artPageHeadings[i].setAttribute('style','border-top-width:0px');</code></p><p><code>593</code><code>		}</code></p><p><code>594</code><code>	}</code></p><p><code>595</code><code>}</code></p><p><code>596</code><code>rmHeaderBorderTop.noBD();</code></p><p><code>597</code><code>&lt;/script&gt;</code></p><p><code>598</code><code>          </code></p><p><code>599</code><code>          </code></p><p><code>600</code><code>          </code></p><p><code>601</code><code>&lt;script&gt;</code></p><p><code>602</code><code>	window.addEventListener("load", pageLoaded, false);</code></p><p><code>603</code><code>	function pageLoaded() {</code></p><p><code>604</code><code>		let tarArtBT = $('#content &gt; div.titleAndBadge');</code></p><p><code>605</code><code>		if (tarArtBT[0].children[0].className === "altmetric-embed altmetric-hidden") {</code></p><p><code>606</code><code>			tarArtBT[0].children[0].nextElementSibling.setAttribute('class', 'tocArticleTitleHidden');</code></p><p><code>607</code><code>		}</code></p><p><code>608</code><code>	}</code></p><p><code>609</code><code>&lt;/script&gt;</code></p><p><code>610</code><code>          </code></p><p><code>611</code><code>          </code></p><p><code>612</code><code>          </code></p><p><code>613</code><code>	&lt;div style="position:relative; top:10px;"&gt;</code></p><p><code>614</code><code>		&lt;div id="trendmd-suggestions"&gt;&lt;/div&gt;		&lt;br /&gt;</code></p><p><code>615</code><code>		&lt;a rel="license" target="_new" href="https://creativecommons.org/licenses/by/3.0/"&gt;&lt;img alt="Creative Commons License" style="border-width:0" src="/images/80x15.png" /&gt;&lt;/a&gt;</code></p><p><code>616</code><code>		All site content, except where otherwise noted, is licensed under a &lt;a target="_blank" rel="license" href="https://creativecommons.org/licenses/by/3.0/"&gt;Creative Commons Attribution 3.0 License&lt;/a&gt;.</code></p><p><code>617</code><code>		&lt;br/&gt;</code></p><p><code>618</code><code>		&lt;span class="article__pii"&gt;</code></p><p><code>619</code><code>							PII: &lt;em&gt;2480&lt;/em&gt;</code></p><p><code>620</code><code>					&lt;/span&gt;</code></p><p><code>621</code><code>	&lt;/div&gt;</code></p><p><code>622</code><code>          </code></p><p><code>623</code><code>&lt;/div&gt;</code></p><p><code>624</code><code>&lt;div style="clear:both;"/&gt;</code></p><p><code>625</code><code>&lt;/div&gt;</code></p><p><code>626</code><code>          </code></p><p><code>627</code><code>	</code></p><p><code>628</code><code>		&lt;script defer src='//js.trendmd.com/trendmd.min.js' data-trendmdconfig='{"journal_id":"76717", "element":"#trendmd-suggestions"}'&gt;&lt;/script&gt;</code></p><p><code>629</code><code>	</code></p><p><code>630</code><code>          </code></p><p><code>631</code><code>	&lt;script type="text/javascript"&gt;</code></p><p><code>632</code><code>		</code></p><p><code>633</code><code>		let tracker = null;</code></p><p><code>634</code><code>		let trenmd_widget = $("#trendmd-suggestions");</code></p><p><code>635</code><code>		let lft_sbar = $("#leftSidebar");</code></p><p><code>636</code><code>		let prev_container_height = 0;</code></p><p><code>637</code><code>	</code></p><p><code>638</code><code>		new MutationObserver(() =&gt; {</code></p><p><code>639</code><code>			let child = $(trenmd_widget).children('div').first();</code></p><p><code>640</code><code>			if (child &amp;&amp; !tracker) {</code></p><p><code>641</code><code>				$(child).find('h3:contains("We recommend")').html("Recommended articles from TrendMD").css('text-transform','none');</code></p><p><code>642</code><code>				tracker = new MutationObserver(() =&gt; {</code></p><p><code>643</code><code>					let container_height = $(child).height();</code></p><p><code>644</code><code>					if (prev_container_height == container_height) { return; }</code></p><p><code>645</code><code>	</code></p><p><code>646</code><code>					$(trenmd_widget).height(container_height);</code></p><p><code>647</code><code>					$(lft_sbar).height($(lft_sbar).height()-prev_container_height);</code></p><p><code>648</code><code>					dispatchEvent(new Event('load'));</code></p><p><code>649</code><code>					prev_container_height = container_height;</code></p><p><code>650</code><code>				}).observe($(child).get(0), { attributes:true, subtree: true });</code></p><p><code>651</code><code>			}</code></p><p><code>652</code><code>		}).observe(document.querySelector('div#trendmd-suggestions'), { childList: true });</code></p><p><code>653</code><code>	</code></p><p><code>654</code><code>	&lt;/script&gt;</code></p><p><code>655</code><code>          </code></p><p><code>656</code><code>&lt;/body&gt;</code></p><p><code>657</code><code>&lt;/html&gt;</code></p>
<div></div></main><iframe hidden="hidden" title="Hidden frame for spell checking" src="about:blank" id="iframeSpelling"></iframe></body></html>